<html lang="en" class=""><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He55s2v847342615za200&amp;tag_exp=101509157~103116026~103200004~103233427~103252644~103252646~103351869~103351871~104481633~104481635~104559073~104559075~104612245~104612247"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            SpatialMap: Spatial Mapping of Unmeasured Gene Expression Profiles in Spatial Transcriptomic Data Using Generalized Linear Spatial Models - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="118E307683B96893023076007A764FF0.m_1">
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov">
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="frontgene">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Frontiers in Genetics">
<meta name="citation_title" content="SpatialMap: Spatial Mapping of Unmeasured Gene Expression Profiles in Spatial Transcriptomic Data Using Generalized Linear Spatial Models">
<meta name="citation_author" content="Dalong Gao">
<meta name="citation_author_institution" content="First Department of Orthopedics, The Second Affiliated Hospital of Xi’an Jiaotong University, Xi’an, China">
<meta name="citation_author_institution" content="Sports Medicine and Joint Surgery, Xianyang Central Hospital, Xianyang, China">
<meta name="citation_author" content="Jin Ning">
<meta name="citation_author_institution" content="School of Public Health, Xi’an Jiaotong University, Xi’an, China">
<meta name="citation_author" content="Gang Liu">
<meta name="citation_author_institution" content="Sports Medicine and Joint Surgery, Xianyang Central Hospital, Xianyang, China">
<meta name="citation_author" content="Shiquan Sun">
<meta name="citation_author_institution" content="School of Public Health, Xi’an Jiaotong University, Xi’an, China">
<meta name="citation_author_institution" content="Key Laboratory of Trace Elements and Endemic Disease, Xi’an Jiaotong University, Xi’an, China">
<meta name="citation_author" content="Xiaoqian Dang">
<meta name="citation_author_institution" content="First Department of Orthopedics, The Second Affiliated Hospital of Xi’an Jiaotong University, Xi’an, China">
<meta name="citation_publication_date" content="2022 May 26">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="893522">
<meta name="citation_doi" content="10.3389/fgene.2022.893522">
<meta name="citation_pmid" content="35692845">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/pdf/fgene-13-893522.pdf">
<meta name="description" content="Recent advances in various single-cell RNA sequencing (scRNA-seq) technologies have enabled profiling the gene expression level with the whole transcriptome at a single-cell resolution. However, it lacks the spatial context of tissues. The ...">
<meta name="og:title" content="SpatialMap: Spatial Mapping of Unmeasured Gene Expression Profiles in Spatial Transcriptomic Data Using Generalized Linear Spatial Models">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Recent advances in various single-cell RNA sequencing (scRNA-seq) technologies have enabled profiling the gene expression level with the whole transcriptome at a single-cell resolution. However, it lacks the spatial context of tissues. The ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script src="/static/assets/mathjax/ui/lazy.js" charset="UTF-8"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/8.571f965df765e09a8d38.chunk.js?Q_CLIENTVERSION=2.29.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><style>.MJX_LiveRegion {
  position: absolute;
  top: 0;
  display: block;
  width: auto;
  height: auto;
  padding: 0;
  opacity: 1;
  z-index: 202;
  left: 0;
  right: 0;
  margin: 0 auto;
  background-color: white;
  box-shadow: 0px 5px 20px #888;
  border: 2px solid #CCCCCC;
}</style><style>.MJX_LiveRegion {
  position: absolute;
  top: 0;
  display: block;
  width: auto;
  height: auto;
  padding: 0;
  opacity: 1;
  z-index: 202;
  left: 0;
  right: 0;
  margin: 0 auto;
  background-color: white;
  box-shadow: 0px 5px 20px #888;
  border: 2px solid #CCCCCC;
}</style><style>.MJX_HoverRegion {
  display: block;
  position: absolute;
  width: max-content;
  height: auto;
  padding: 0;
  opacity: 1;
  z-index: 202;
  margin: 0 auto;
  background-color: white;
  line-height: 0;
  box-shadow: 0px 10px 20px #888;
  border: 2px solid #CCCCCC;
}

.MJX_HoverRegion > div {
  overflow: hidden;
}</style><style>.MJX_ToolTip {
  width: auto;
  height: auto;
  opacity: 1;
  text-align: center;
  border-radius: 6px;
  padding: 0;
  border-bottom: 1px dotted black;
  position: absolute;
  display: inline-block;
  background-color: white;
  z-index: 202;
}</style><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container[jax="CHTML"] :focus {
  outline: solid 3px;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("/static/assets/mathjax/chtml/woff/mjx-mm-em.woff") format("woff");
}
</style><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.dced7c59e38f6c5e287c.chunk.js?Q_CLIENTVERSION=2.29.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.f285b832c657f9989ab7.chunk.js?Q_CLIENTVERSION=2.29.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="9181802">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <span class="usa-tooltip"><a href="https://doi.org/10.3389/fgene.2022.893522" class="usa-link display-flex usa-tooltip__trigger" role="button" target="_blank" rel="noreferrer noopener" data-position="bottom" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile" aria-describedby="tooltip-704497" tabindex="0">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a><span class="usa-tooltip__body" id="tooltip-704497" role="tooltip" aria-hidden="true">View on publisher site</span></span>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <span class="usa-tooltip"><a href="pdf/fgene-13-893522.pdf" class="usa-link display-flex usa-tooltip__trigger" role="button" data-position="bottom" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile" aria-describedby="tooltip-616629" tabindex="0">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a><span class="usa-tooltip__body usa-tooltip__body--bottom" id="tooltip-616629" role="tooltip" aria-hidden="true" style="left: 50%; margin: 5px 0px 0px -57px;">Download PDF</span></span>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <span class="usa-tooltip"><button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty usa-tooltip__trigger" data-position="bottom" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC9181802%2F%23open-collections-dialog" data-in-collections="false" aria-describedby="tooltip-475051" tabindex="0">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button><span class="usa-tooltip__body usa-tooltip__body--bottom" id="tooltip-475051" role="tooltip" aria-hidden="true" style="left: 50%; margin: 5px 0px 0px -68.5px;">Add to Collections</span></span>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <span class="usa-tooltip"><button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex usa-tooltip__trigger" aria-label="Open dialog with citation text in different styles" data-position="bottom" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/9181802/" data-citation-style="nlm" data-download-format-link="/resources/citations/9181802/export/" aria-describedby="tooltip-928799" tabindex="0">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button><span class="usa-tooltip__body usa-tooltip__body--bottom is-visible" id="tooltip-928799" role="tooltip" aria-hidden="true" style="left: 50%; margin: 5px 0px 0px -21px;">Cite</span></span>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <span class="usa-tooltip"><button type="button" data-position="bottom" class="usa-button usa-button--unstyled display-flex usa-tooltip__trigger" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile" aria-describedby="tooltip-789272" tabindex="0">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button><span class="usa-tooltip__body usa-tooltip__body--left is-visible" id="tooltip-789272" role="tooltip" aria-hidden="true" style="top: 50%; left: -5px; margin: -20px 0px 0px -84px;">Permalink</span></span>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-frontgene.gif" alt="Frontiers in Genetics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Frontiers in Genetics" title="Link to Frontiers in Genetics" shape="default" href="http://www.frontiersin.org/genetics" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Front Genet</button></div>. 2022 May 26;13:893522. doi: <a href="https://doi.org/10.3389/fgene.2022.893522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.3389/fgene.2022.893522</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Front%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Front%20Genet%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Front%20Genet%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Front%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>SpatialMap: Spatial Mapping of Unmeasured Gene Expression Profiles in Spatial Transcriptomic Data Using Generalized Linear Spatial Models</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Dalong Gao</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Dalong Gao</span></h3>
<div class="p">
<sup>1</sup>
First Department of Orthopedics, The Second Affiliated Hospital of Xi’an Jiaotong University, Xi’an, China
</div>
<div class="p">
<sup>2</sup>
Sports Medicine and Joint Surgery, Xianyang Central Hospital, Xianyang, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dalong Gao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ning%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Jin Ning</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jin Ning</span></h3>
<div class="p">
<sup>3</sup>
School of Public Health, Xi’an Jiaotong University, Xi’an, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ning%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jin Ning</span></a>
</div>
</div>
<sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Gang Liu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Gang Liu</span></h3>
<div class="p">
<sup>2</sup>
Sports Medicine and Joint Surgery, Xianyang Central Hospital, Xianyang, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gang Liu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Shiquan Sun</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shiquan Sun</span></h3>
<div class="p">
<sup>3</sup>
School of Public Health, Xi’an Jiaotong University, Xi’an, China
</div>
<div class="p">
<sup>4</sup>
Key Laboratory of Trace Elements and Endemic Disease, Xi’an Jiaotong University, Xi’an, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiquan Sun</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Xiaoqian Dang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xiaoqian Dang</span></h3>
<div class="p">
<sup>1</sup>
First Department of Orthopedics, The Second Affiliated Hospital of Xi’an Jiaotong University, Xi’an, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoqian Dang</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
First Department of Orthopedics, The Second Affiliated Hospital of Xi’an Jiaotong University, Xi’an, China
</div>
<div id="aff2">
<sup>2</sup>
Sports Medicine and Joint Surgery, Xianyang Central Hospital, Xianyang, China
</div>
<div id="aff3">
<sup>3</sup>
School of Public Health, Xi’an Jiaotong University, Xi’an, China
</div>
<div id="aff4">
<sup>4</sup>
Key Laboratory of Trace Elements and Endemic Disease, Xi’an Jiaotong University, Xi’an, China
</div>
<div class="author-notes p">
<div class="fn" id="fn1_pmc"><p>
<strong>Edited by:</strong>
<a href="https://loop.frontiersin.org/people/800132/overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tao Wang</a>, Northwestern Polytechnical University, China</p></div>
<div class="fn" id="fn2">
<p>
<strong>Reviewed by:</strong>
<a href="https://loop.frontiersin.org/people/599236/overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Audrey Qiuyan Fu</a>, University of Idaho, United States</p>
<p>
<a href="https://loop.frontiersin.org/people/895157/overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Sheng Yang</a>, Nanjing Medical University, China</p>
</div>
<div class="fn" id="c001">
<sup>✉</sup><p class="display-inline">*Correspondence: Shiquan Sun, <span>sqsunsph@xjtu.edu.cn</span>; Xiaoqian Dang, <span>dangxiaoqian@vip.163.com</span>
</p>
</div>
<div class="fn" id="fn1">
<sup>
<sup>†</sup>
</sup><p class="display-inline">These authors have contributed equally to this work</p>
</div>
<div class="fn" id="fn4"><p>This article was submitted to Statistical Genetics and Methodology, a section of the journal Frontiers in Genetics</p></div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2022 Mar 10; Accepted 2022 Apr 19; Collection date 2022.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2022 Gao, Ning, Liu, Sun and Dang.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC9181802&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/35692845/" class="usa-link">35692845</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2 data-anchor-id="abstract1"><a id="abstract1-anchor" data-anchor-id="abstract1" class="usa-anchor"></a>Abstract</h2>
<p>Recent advances in various single-cell RNA sequencing (scRNA-seq) technologies have enabled profiling the gene expression level with the whole transcriptome at a single-cell resolution. However, it lacks the spatial context of tissues. The image-based transcriptomics <em>in situ</em> studies (e.g., MERFISH and seqFISH) maintain the cell spatial context at individual cell levels but can only measure a limited number of genes or transcripts (up to roughly 1,000 genes). Therefore, integrating scRNA-seq data and image-based transcriptomics data can potentially gain the complementary benefits of both. Here, we develop a computational method, SpatialMap, to bridge the gap, which primarily facilitates spatial mapping of unmeasured gene profiles in spatial transcriptomic data <em>via</em> integrating with scRNA-seq data from the same tissue. SpatialMap directly models the count nature of spatial gene expression data through generalized linear spatial models, which accounts for the spatial correlation among spatial locations using conditional autoregressive (CAR) prior. With a newly developed computationally efficient penalized quasi-likelihood (PQL)-based algorithm, SpatialMap can scale up to performing large-scale spatial mapping analysis. Finally, we applied the SpatialMap to four publicly available tissue-paired studies (i.e., scRNA-seq studies and image-based transcriptomics studies). The results demonstrate that the proposed method can accurately predict unmeasured gene expression profiles across various spatial and scRNA-seq dataset pairs of different species and technologies.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> scRNA-seq, spatial transcriptomics, generalized linear spatial model, integrative analysis, penalized quasi-likelihood</p></section></section><section id="s1"><h2 class="pmc_sec_title" data-anchor-id="s1"><a id="s1-anchor" data-anchor-id="s1" class="usa-anchor"></a>1 Introduction</h2>
<p>Single-cell RNA sequencing (scRNA-seq) is becoming an established experimental technique for transcriptome profiling to study the functional states of individual cells and to reveal all possible cell types/states (<a href="#B30" class="usa-link" aria-describedby="B30" aria-expanded="false">Tim and Satija, 2019</a>; <a href="#B17" class="usa-link" aria-describedby="B17" aria-expanded="false">Ma et al., 2020</a>; <a href="#B14" class="usa-link" aria-describedby="B14" aria-expanded="false">Lindeboom et al., 2021</a>). Current scRNA-seq technologies/protocols allow whole-transcriptome sequencing in an unbiased manner (<a href="#B20" class="usa-link" aria-describedby="B20" aria-expanded="false">Picelli et al., 2014</a>; <a href="#B35" class="usa-link" aria-describedby="B35" aria-expanded="false">Zheng et al., 2017</a>; <a href="#B18" class="usa-link" aria-describedby="B18" aria-expanded="false">Mereu et al., 2020</a>). However, these protocols commonly result in the dissociation of tissue of origin to single cells and therefore loss of spatial context. Since the isolated cells are out of their original tissue matrix or natural microenvironment, altered transcriptional states do not truly reflect the natural states of a cell within the tissue of origin (<a href="#B24" class="usa-link" aria-describedby="B24" aria-expanded="false">Ståhl Patrik et al., 2016</a>; <a href="#B5" class="usa-link" aria-describedby="B5" aria-expanded="false">Burgess, 2019</a>; <a href="#B28" class="usa-link" aria-describedby="B28" aria-expanded="false">Sun et al., 2020</a>; <a href="#B36" class="usa-link" aria-describedby="B36" aria-expanded="false">Zhu et al., 2021</a>), which is a crucial step toward understanding spatial heterogeneity of complex tissues (<a href="#B3" class="usa-link" aria-describedby="B3" aria-expanded="false">Antebi et al., 2017</a>; <a href="#B12" class="usa-link" aria-describedby="B12" aria-expanded="false">Levy-Jurgenson et al., 2020</a>; <a href="#B22" class="usa-link" aria-describedby="B22" aria-expanded="false">Saviano et al., 2020</a>) ].</p>
<p>Single-molecule fluorescence <em>in situ</em> hybridization (smFISH) or image-based transcriptomics <em>in situ</em> studies (<a href="#B4" class="usa-link" aria-describedby="B4" aria-expanded="false">Battich et al., 2013</a>) provide an unprecedented opportunity to decipher the functional states on the spatial landscape at single-cell or even subcellular resolution within the context of tissue of origin, such as MERFISH (<a href="#B33" class="usa-link" aria-describedby="B33" aria-expanded="false">Xia et al., 2019</a>) and seqFISH (<a href="#B23" class="usa-link" aria-describedby="B23" aria-expanded="false">Shah et al., 2018</a>). However, these protocols usually only assay a limited number of genes or transcripts and therefore biased transcriptome profiling of one particular cell. Although a recent evolution of sequential fluorescence <em>in situ</em> hybridization (e.g., seqFISH+) achieves super-resolution imaging and multiplexing of genes up to around 10,000 per cell (<a href="#B15" class="usa-link" aria-describedby="B15" aria-expanded="false">Linus Eng et al., 2019</a>; <a href="#B33" class="usa-link" aria-describedby="B33" aria-expanded="false">Xia et al., 2019</a>), it always involves the heavy computational burden with increasing data storage (<a href="#B25" class="usa-link" aria-describedby="B25" aria-expanded="false">Strell et al., 2019</a>).</p>
<p>Overall, scRNA-seq technologies allow whole-transcriptome profiling but for the loss of spatial context, while image-based transcriptomics <em>in situ</em> studies maintain spatial context but profile a limited number of genes. Therefore, to bridge such a gap, an integrative analysis of scRNA-seq studies and image-based transcriptomics <em>in situ</em> studies is potentially able to gain complimentary benefits from both of them. Recently, many integrative methods were developed to map the single cells dissociated from scRNA-seq protocols to the spatial landscape of the same tissues of origin (<a href="#B10" class="usa-link" aria-describedby="B10" aria-expanded="false">James et al., 2014</a>; <a href="#B2" class="usa-link" aria-describedby="B2" aria-expanded="false">Achim et al., 2015</a>; <a href="#B21" class="usa-link" aria-describedby="B21" aria-expanded="false">Satija et al., 2015</a>), while only one machine learning method, SpaGE, was developed for predicting the missing gene expression measurements in the spatial data <em>via</em> integrating with scRNA-seq data from the same tissue (<a href="#B1" class="usa-link" aria-describedby="B1" aria-expanded="false">Abdelaal et al., 2020</a>). SpaGE integrates spatial and scRNA-seq datasets to predict unmeasured expression profiles in their spatial configuration. On the one hand, however, SpaGE requires the data transformation step of both data prior to performing the integrative analysis. However, it is documented in many other studies that analyzing normalized data can be suboptimal as this approach fails to account for the mean–variance relationship that existed in raw counts (<a href="#B26" class="usa-link" aria-describedby="B26" aria-expanded="false">Sun et al., 2017</a>; <a href="#B16" class="usa-link" aria-describedby="B16" aria-expanded="false">Lun, 2018</a>). On the other hand, SpaGE ignores the spatial correlation among spatial locations and therefore lost the spatial heterogeneity modeling in the analysis.</p>
<p>In this article, we develop SpatialMap, a robust statistical method to predict unmeasured gene expression profiles of each cell in spatial transcriptomic data through the integration with scRNA-seq data from the same tissue. SpatialMap relies on the generalized linear spatial models that directly model the count nature of gene expression levels of spatial transcriptomics data and accounts for the spatial heterogeneity using conditional autoregressive (CAR) prior (<a href="#B11" class="usa-link" aria-describedby="B11" aria-expanded="false">Julian and Kooperberg, 1995</a>). Moreover, we develop a computationally efficient algorithm based on the penalized quasi-likelihood (PQL) algorithm (<a href="#B27" class="usa-link" aria-describedby="B27" aria-expanded="false">Sun et al., 2019</a>). Finally, we apply SpatialMap on four tissue-paired datasets consisting of scRNA-seq data and image-based spatial transcriptomics data on the same tissue. The results demonstrate that SpatialMap can accurately predict unmeasured gene expression profiles across various spatial and scRNA-seq dataset pairs and would be, especially conducive to spatial downstream analysis when the image-based spatial data are poor in transcript profiles.</p></section><section id="s2"><h2 class="pmc_sec_title" data-anchor-id="s2"><a id="s2-anchor" data-anchor-id="s2" class="usa-anchor"></a>2 Results</h2>
<section id="s2-1"><h3 class="pmc_sec_title">2.1 SpatialMap Overview</h3>
<p>We developed a computational method, SpatialMap, to predict unmeasured gene expression levels of the spatial landscape using scRNA-seq data (<a href="#F1" class="usa-link">Figure 1A</a>). We briefly provide an overview of SpatialMap; more details are available in Methods and Materials. We take spatial and scRNA-seq gene expression matrices as inputs; to characterize spatially expressed patterns with genes, we examine one gene at a time. For each gene measured by image-based technologies, we model the raw read counts with a Poisson distribution, i.e.,</p>
<table class="disp-formula p" id="e1"><tbody><tr>
<td class="formula"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="linebreak" display="true" tabindex="0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="17" style="font-size: 103.8%;"><mjx-math id="m1" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-annotation="depth:1" data-semantic-id="35" data-semantic-children="33,18,34,23,24,25,26,27" data-semantic-content="18,23,25,27" data-semantic-owns="33 18 34 23 24 25 26 27" aria-level="0" aria-posinset="1" aria-setsize="1" role="treeitem" aria-owns="33 18 34 23 24 25 26 27" data-semantic-structure="(35 (33 (2 0 1) 3 (32 4 30 5 31 (29 6 28 (17 15 (14 (9 7 8) 13 (12 10 11)) 16)))) 18 (34 20 21 22) 23 24 25 26 27)" display="true" class="MM-N" style="margin-left: 0px; margin-right: 0px;"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-annotation="depth:2" data-semantic-id="33" data-semantic-children="2,32" data-semantic-content="3" data-semantic-parent="35" data-semantic-owns="2 3 32" aria-level="1" aria-posinset="1" aria-setsize="8" role="treeitem" aria-owns="2 3 32"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-annotation="depth:3" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="33" data-semantic-owns="0 1" aria-level="2" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:4" data-semantic-id="0" data-semantic-parent="2" aria-level="3" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D466">𝑦</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:4" data-semantic-id="1" data-semantic-parent="2" aria-level="3" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="33" data-semantic-operator="relseq,∼" aria-level="2" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:3" data-semantic-id="32" data-semantic-children="4,5,29" data-semantic-content="30,31" data-semantic-parent="33" data-semantic-owns="4 30 5 31 29" aria-level="2" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="4 30 5 31 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:4" data-semantic-id="4" data-semantic-parent="32" aria-level="3" aria-posinset="1" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="3" aria-posinset="2" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:4" data-semantic-id="5" data-semantic-parent="32" aria-level="3" aria-posinset="3" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="3" aria-posinset="4" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="depth:4" data-semantic-id="29" data-semantic-children="6,17" data-semantic-content="28,6" data-semantic-parent="32" data-semantic-owns="6 28 17" aria-level="3" aria-posinset="5" aria-setsize="5" role="treeitem" aria-owns="6 28 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:5" data-semantic-id="6" data-semantic-parent="29" data-semantic-operator="appl" aria-level="4" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="appl" aria-level="4" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-annotation="depth:5" data-semantic-id="17" data-semantic-children="14" data-semantic-content="15,16" data-semantic-parent="29" data-semantic-owns="15 14 16" aria-level="4" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="15 14 16"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="fenced" aria-level="5" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:6" data-semantic-id="14" data-semantic-children="9,12" data-semantic-content="13" data-semantic-parent="17" data-semantic-owns="9 13 12" aria-level="5" aria-posinset="2" aria-setsize="3" role="treeitem" aria-owns="9 13 12"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-annotation="depth:7" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="14" data-semantic-owns="7 8" aria-level="6" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="7 8"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:8" data-semantic-id="7" data-semantic-parent="9" aria-level="7" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:8" data-semantic-id="8" data-semantic-parent="9" aria-level="7" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="6" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-annotation="depth:7" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="14" data-semantic-owns="10 11" aria-level="6" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="10 11"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:8" data-semantic-id="10" data-semantic-parent="12" aria-level="7" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:8" data-semantic-id="11" data-semantic-parent="12" aria-level="7" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="fenced" aria-level="5" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:2" data-semantic-id="18" data-semantic-parent="35" data-semantic-operator="punctuated" aria-level="1" aria-posinset="2" aria-setsize="8" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mspace style="width: 0.333em;"></mjx-mspace><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-annotation="depth:2" data-semantic-id="34" data-semantic-children="20,22" data-semantic-content="21" data-semantic-parent="35" data-semantic-owns="20 21 22" aria-level="1" aria-posinset="3" aria-setsize="8" role="treeitem" aria-owns="20 21 22"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:3" data-semantic-id="20" data-semantic-parent="34" aria-level="2" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="21" data-semantic-parent="34" data-semantic-operator="relseq,=" aria-level="2" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:3" data-semantic-id="22" data-semantic-parent="34" aria-level="2" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.666em;">1</mjx-c><mjx-c class="mjx-c2C" noic="true" style="padding-top: 0.666em;">,</mjx-c><mjx-c class="mjx-c32" style="padding-top: 0.666em;">2</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="nemeth:number;depth:2" data-semantic-id="23" data-semantic-parent="35" data-semantic-operator="punctuated" aria-level="1" aria-posinset="4" aria-setsize="8" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-annotation="nemeth:number;depth:2" data-semantic-id="24" data-semantic-parent="35" aria-level="1" aria-posinset="5" aria-setsize="8" role="treeitem"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="nemeth:number;depth:2" data-semantic-id="25" data-semantic-parent="35" data-semantic-operator="punctuated" aria-level="1" aria-posinset="6" aria-setsize="8" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:2" data-semantic-id="26" data-semantic-parent="35" aria-level="1" aria-posinset="7" aria-setsize="8" role="treeitem"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="nemeth:number;depth:2" data-semantic-id="27" data-semantic-parent="35" data-semantic-operator="punctuated" aria-level="1" aria-posinset="8" aria-setsize="8" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></td>
<td class="label">(1)</td>
</tr></tbody></table>
<p>where <em>Poi</em>(⋅) is the Poisson distribution, <em>N<sub>i</sub>
</em> is the summation of the total number of counts across all genes for spatial spot <em>i</em>, and <em>λ</em>
<sub>
<em>i</em>
</sub> is an unknown Poisson rate parameter, which can be modeled as the linear combination of the following terms:</p>
<table class="disp-formula p" id="e2"><tbody><tr>
<td class="formula"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="linebreak" display="true" tabindex="0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="18" style="font-size: 103.8%;"><mjx-math id="m2" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-annotation="depth:1" data-semantic-id="29" data-semantic-children="28,22" data-semantic-content="22" data-semantic-owns="28 22" aria-level="0" aria-posinset="1" aria-setsize="1" role="treeitem" aria-owns="28 22" data-semantic-structure="(29 (28 (24 0 23 (6 4 (3 1 2) 5)) 7 (27 8 9 (26 (12 10 11) 25 13) 14 (17 15 16) 18 (21 19 20))) 22)" display="true" class="MM-N" style="margin-left: 0px; margin-right: 0px;"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-annotation="depth:2" data-semantic-id="28" data-semantic-children="24,27" data-semantic-content="7" data-semantic-parent="29" data-semantic-owns="24 7 27" aria-level="1" aria-posinset="1" aria-setsize="2" role="treeitem" aria-owns="24 7 27"><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-annotation="depth:3" data-semantic-id="24" data-semantic-children="0,6" data-semantic-content="23,0" data-semantic-parent="28" data-semantic-owns="0 23 6" aria-level="2" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="0 23 6"><mjx-mi data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-annotation="nemeth:number;depth:4" data-semantic-id="0" data-semantic-parent="24" data-semantic-operator="appl" aria-level="3" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.694em;">g</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="appl" aria-level="3" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-annotation="depth:4" data-semantic-id="6" data-semantic-children="3" data-semantic-content="4,5" data-semantic-parent="24" data-semantic-owns="4 3 5" aria-level="3" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="4 3 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="fenced" aria-level="4" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-annotation="depth:5" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="6" data-semantic-owns="1 2" aria-level="4" aria-posinset="2" aria-setsize="3" role="treeitem" aria-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:6" data-semantic-id="1" data-semantic-parent="3" aria-level="5" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:6" data-semantic-id="2" data-semantic-parent="3" aria-level="5" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="fenced" aria-level="4" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="7" data-semantic-parent="28" data-semantic-operator="relseq,=" aria-level="2" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-annotation="depth:3" data-semantic-id="27" data-semantic-children="8,26,17,21" data-semantic-content="9,14,18" data-semantic-parent="28" data-semantic-owns="8 9 26 14 17 18 21" aria-level="2" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="8 9 26 14 17 18 21"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:4" data-semantic-id="8" data-semantic-parent="27" aria-level="3" aria-posinset="1" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="9" data-semantic-parent="27" data-semantic-operator="infixop,+" aria-level="3" aria-posinset="2" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:4" data-semantic-id="26" data-semantic-children="12,13" data-semantic-content="25" data-semantic-parent="27" data-semantic-owns="12 25 13" aria-level="3" aria-posinset="3" aria-setsize="7" role="treeitem" aria-owns="12 25 13"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-annotation="depth:5" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="26" data-semantic-owns="10 11" aria-level="4" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="10 11"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:6" data-semantic-id="10" data-semantic-parent="12" aria-level="5" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:6" data-semantic-id="11" data-semantic-parent="12" aria-level="5" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="4" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:5" data-semantic-id="13" data-semantic-parent="26" aria-level="4" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="27" data-semantic-operator="infixop,+" aria-level="3" aria-posinset="4" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-annotation="depth:4" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="27" data-semantic-owns="15 16" aria-level="3" aria-posinset="5" aria-setsize="7" role="treeitem" aria-owns="15 16"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:5" data-semantic-id="15" data-semantic-parent="17" aria-level="4" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:5" data-semantic-id="16" data-semantic-parent="17" aria-level="4" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="18" data-semantic-parent="27" data-semantic-operator="infixop,+" aria-level="3" aria-posinset="6" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-annotation="depth:4" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="27" data-semantic-owns="19 20" aria-level="3" aria-posinset="7" aria-setsize="7" role="treeitem" aria-owns="19 20"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:5" data-semantic-id="19" data-semantic-parent="21" aria-level="4" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:5" data-semantic-id="20" data-semantic-parent="21" aria-level="4" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="nemeth:number;depth:2" data-semantic-id="22" data-semantic-parent="29" data-semantic-operator="punctuated" aria-level="1" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></td>
<td class="label">(2)</td>
</tr></tbody></table>
<p>where <em>α</em> is an intercept; <em>x</em>
<sub>
<em>i</em>
</sub> refers to the cell-type-specific expression-level computed by scRNA-seq for the <em>i</em>th pixel, and <em>β</em> is its corresponding coefficient; <em>e</em>
<sub>
<em>i</em>
</sub> is the residual term that is independently and identically distributed following <em>N</em> (0, <em>σ</em>
<sup>2</sup> (1 − <em>h</em>
<sup>2</sup>)), with <em>σ</em>
<sub>2</sub> and <em>h</em>
<sup>2</sup> being the scaling factors; and <em>g</em>
<sub>
<em>i</em>
</sub> is a random effect modeling the spatial correlation pattern among spatial pixels with kernel functions, i.e.,</p>
<table class="disp-formula p" id="e3"><tbody><tr>
<td class="formula"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="linebreak" display="true" tabindex="0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="19" style="font-size: 103.8%;"><mjx-math id="m3" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-annotation="depth:1" data-semantic-id="49" data-semantic-children="48,42" data-semantic-content="42" data-semantic-owns="48 42" aria-level="0" aria-posinset="1" aria-setsize="1" role="treeitem" aria-owns="48 42" data-semantic-structure="(49 (48 0 1 (20 (18 16 (15 (4 2 3) 5 (8 6 7) 9 10 11 (14 12 13)) 17) 19) 21 (47 22 45 23 46 (44 24 43 (41 39 (38 25 26 (37 (29 27 28) 36 (35 (32 30 31) 34 33))) 40)))) 42)" display="true" class="MM-N" style="margin-left: 0px; margin-right: 0px;"><mjx-mrow data-semantic-added="true" data-semantic-type="multirel" data-semantic-role="equality" data-semantic-annotation="depth:2" data-semantic-id="48" data-semantic-children="0,20,47" data-semantic-content="1,21" data-semantic-parent="49" data-semantic-owns="0 1 20 21 47" aria-level="1" aria-posinset="1" aria-setsize="2" role="treeitem" aria-owns="0 1 20 21 47"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:3" data-semantic-id="0" data-semantic-parent="48" aria-level="2" aria-posinset="1" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="48" data-semantic-operator="multirel" aria-level="2" aria-posinset="2" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msup space="4" data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-annotation="depth:3" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="48" data-semantic-owns="18 19" aria-level="2" aria-posinset="3" aria-setsize="5" role="treeitem" aria-owns="18 19"><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-annotation="depth:4" data-semantic-id="18" data-semantic-children="15" data-semantic-content="16,17" data-semantic-parent="20" data-semantic-owns="16 15 17" aria-level="3" aria-posinset="1" aria-setsize="2" role="treeitem" aria-owns="16 15 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="fenced" aria-level="4" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-annotation="depth:5" data-semantic-id="15" data-semantic-children="4,5,8,9,10,11,14" data-semantic-content="5,9,11" data-semantic-parent="18" data-semantic-owns="4 5 8 9 10 11 14" aria-level="4" aria-posinset="2" aria-setsize="3" role="treeitem" aria-owns="4 5 8 9 10 11 14"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-annotation="depth:6" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="15" data-semantic-owns="2 3" aria-level="5" aria-posinset="1" aria-setsize="7" role="treeitem" aria-owns="2 3"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="2" data-semantic-parent="4" aria-level="6" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="3" data-semantic-parent="4" aria-level="6" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:6" data-semantic-id="5" data-semantic-parent="15" data-semantic-operator="punctuated" aria-level="5" aria-posinset="2" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-annotation="depth:6" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="15" data-semantic-owns="6 7" aria-level="5" aria-posinset="3" aria-setsize="7" role="treeitem" aria-owns="6 7"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="6" data-semantic-parent="8" aria-level="6" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="7" data-semantic-parent="8" aria-level="6" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:6" data-semantic-id="9" data-semantic-parent="15" data-semantic-operator="punctuated" aria-level="5" aria-posinset="4" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-annotation="depth:6" data-semantic-id="10" data-semantic-parent="15" aria-level="5" aria-posinset="5" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:6" data-semantic-id="11" data-semantic-parent="15" data-semantic-operator="punctuated" aria-level="5" aria-posinset="6" aria-setsize="7" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-annotation="depth:6" data-semantic-id="14" data-semantic-children="12,13" data-semantic-parent="15" data-semantic-owns="12 13" aria-level="5" aria-posinset="7" aria-setsize="7" role="treeitem" aria-owns="12 13"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="12" data-semantic-parent="14" aria-level="6" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="13" data-semantic-parent="14" aria-level="6" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="fenced" aria-level="4" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:4" data-semantic-id="19" data-semantic-parent="20" aria-level="3" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="21" data-semantic-parent="48" data-semantic-operator="multirel" aria-level="2" aria-posinset="4" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:3" data-semantic-id="47" data-semantic-children="22,23,44" data-semantic-content="45,46" data-semantic-parent="48" data-semantic-owns="22 45 23 46 44" aria-level="2" aria-posinset="5" aria-setsize="5" role="treeitem" aria-owns="22 45 23 46 44"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:4" data-semantic-id="22" data-semantic-parent="47" aria-level="3" aria-posinset="1" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="3" aria-posinset="2" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:4" data-semantic-id="23" data-semantic-parent="47" aria-level="3" aria-posinset="3" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="46" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="3" aria-posinset="4" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="depth:4" data-semantic-id="44" data-semantic-children="24,41" data-semantic-content="43,24" data-semantic-parent="47" data-semantic-owns="24 43 41" aria-level="3" aria-posinset="5" aria-setsize="5" role="treeitem" aria-owns="24 43 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:5" data-semantic-id="24" data-semantic-parent="44" data-semantic-operator="appl" aria-level="4" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="appl" aria-level="4" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-annotation="depth:5" data-semantic-id="41" data-semantic-children="38" data-semantic-content="39,40" data-semantic-parent="44" data-semantic-owns="39 38 40" aria-level="4" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="39 38 40"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="39" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="fenced" aria-level="5" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-annotation="depth:6" data-semantic-id="38" data-semantic-children="25,26,37" data-semantic-content="26" data-semantic-parent="41" data-semantic-owns="25 26 37" aria-level="5" aria-posinset="2" aria-setsize="3" role="treeitem" aria-owns="25 26 37"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="25" data-semantic-parent="38" aria-level="6" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:7" data-semantic-id="26" data-semantic-parent="38" data-semantic-operator="punctuated" aria-level="6" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:7" data-semantic-id="37" data-semantic-children="29,35" data-semantic-content="36" data-semantic-parent="38" data-semantic-owns="29 36 35" aria-level="6" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="29 36 35"><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-annotation="depth:8" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="37" data-semantic-owns="27 28" aria-level="7" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="27 28"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:9" data-semantic-id="27" data-semantic-parent="29" aria-level="8" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.413em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:9" data-semantic-id="28" data-semantic-parent="29" aria-level="8" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="37" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="7" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple;depth:8" data-semantic-id="35" data-semantic-children="32,33" data-semantic-content="34,30" data-semantic-parent="37" data-semantic-owns="32 34 33" aria-level="7" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="32 34 33"><mjx-msup data-semantic-type="superscript" data-semantic-role="simple function" data-semantic-annotation="depth:9" data-semantic-id="32" data-semantic-children="30,31" data-semantic-parent="35" data-semantic-owns="30 31" aria-level="8" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="30 31"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:10" data-semantic-id="30" data-semantic-parent="32" data-semantic-operator="appl" aria-level="9" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.413em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:10" data-semantic-id="31" data-semantic-parent="32" aria-level="9" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="appl" aria-level="8" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:9" data-semantic-id="33" data-semantic-parent="35" aria-level="8" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D43E">𝐾</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="40" data-semantic-parent="41" data-semantic-added="true" data-semantic-operator="fenced" aria-level="5" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="nemeth:number;depth:2" data-semantic-id="42" data-semantic-parent="49" data-semantic-operator="punctuated" aria-level="1" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></td>
<td class="label">(3)</td>
</tr></tbody></table>
<p>where <em>MVN</em> (⋅) denotes the multivariate normal distribution and the variance–covariance matrix <em>K</em> is a kernel function of the <em>n</em> spatial pixels. We set <em>h</em>
<sup>2</sup> ∈ [0, 1], explaining the expression variance in log (<em>λ</em>
<sub>
<em>i</em>
</sub>) owing to the random noise <em>g</em>. <em>e</em> is a <em>n</em>-vector following <em>MVN</em> (0, <em>σ</em>
<sup>2</sup> (1 − <em>h</em>
<sup>2</sup>)<em>I</em>) with <em>I</em> being an <em>n</em> by <em>n</em> identity matrix that models an independent sequencing error. In addition, we also use conditional autoregressive (CAR) models as the prior distributions for spatial random effects with spatial data. By Brook’s Lemma, the joint distribution of spatial random effects can be modified as:</p>
<table class="disp-formula p" id="e4"><tbody><tr>
<td class="formula"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="linebreak" display="true" tabindex="0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="20" style="font-size: 103.8%;"><mjx-math id="m4" data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-annotation="depth:1" data-semantic-id="44" data-semantic-children="43,37" data-semantic-content="37" data-semantic-owns="43 37" aria-level="0" aria-posinset="1" aria-setsize="1" role="treeitem" aria-owns="43 37" data-semantic-structure="(44 (43 0 1 (42 2 40 3 41 (39 4 38 (36 34 (33 5 6 (32 (9 7 8) 31 (30 (12 10 11) 29 (28 (24 22 (21 13 14 (20 (17 15 16) 19 18)) 23) (27 25 26))))) 35)))) 37)" display="true" class="MM-N" style="margin-left: 0px; margin-right: 0px;"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-annotation="depth:2" data-semantic-id="43" data-semantic-children="0,42" data-semantic-content="1" data-semantic-parent="44" data-semantic-owns="0 1 42" aria-level="1" aria-posinset="1" aria-setsize="2" role="treeitem" aria-owns="0 1 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:3" data-semantic-id="0" data-semantic-parent="43" aria-level="2" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="43" data-semantic-operator="relseq,∼" aria-level="2" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:3" data-semantic-id="42" data-semantic-children="2,3,39" data-semantic-content="40,41" data-semantic-parent="43" data-semantic-owns="2 40 3 41 39" aria-level="2" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="2 40 3 41 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:4" data-semantic-id="2" data-semantic-parent="42" aria-level="3" aria-posinset="1" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="40" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="3" aria-posinset="2" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:4" data-semantic-id="3" data-semantic-parent="42" aria-level="3" aria-posinset="3" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="3" aria-posinset="4" aria-setsize="5" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="depth:4" data-semantic-id="39" data-semantic-children="4,36" data-semantic-content="38,4" data-semantic-parent="42" data-semantic-owns="4 38 36" aria-level="3" aria-posinset="5" aria-setsize="5" role="treeitem" aria-owns="4 38 36"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:5" data-semantic-id="4" data-semantic-parent="39" data-semantic-operator="appl" aria-level="4" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="38" data-semantic-parent="39" data-semantic-added="true" data-semantic-operator="appl" aria-level="4" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-annotation="depth:5" data-semantic-id="36" data-semantic-children="33" data-semantic-content="34,35" data-semantic-parent="39" data-semantic-owns="34 33 35" aria-level="4" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="34 33 35"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="34" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="fenced" aria-level="5" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c28 MM-S3">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-annotation="depth:6" data-semantic-id="33" data-semantic-children="5,6,32" data-semantic-content="6" data-semantic-parent="36" data-semantic-owns="5 6 32" aria-level="5" aria-posinset="2" aria-setsize="3" role="treeitem" aria-owns="5 6 32"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:7" data-semantic-id="5" data-semantic-parent="33" aria-level="6" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:7" data-semantic-id="6" data-semantic-parent="33" data-semantic-operator="punctuated" aria-level="6" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:7" data-semantic-id="32" data-semantic-children="9,30" data-semantic-content="31" data-semantic-parent="33" data-semantic-owns="9 31 30" aria-level="6" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="9 31 30"><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-annotation="depth:8" data-semantic-id="9" data-semantic-children="7,8" data-semantic-parent="32" data-semantic-owns="7 8" aria-level="7" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="7 8"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:9" data-semantic-id="7" data-semantic-parent="9" aria-level="8" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D70E">𝜎</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.413em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:9" data-semantic-id="8" data-semantic-parent="9" aria-level="8" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="7" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="depth:8" data-semantic-id="30" data-semantic-children="12,28" data-semantic-content="29,10" data-semantic-parent="32" data-semantic-owns="12 29 28" aria-level="7" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="12 29 28"><mjx-msup data-semantic-type="superscript" data-semantic-role="simple function" data-semantic-annotation="depth:9" data-semantic-id="12" data-semantic-children="10,11" data-semantic-parent="30" data-semantic-owns="10 11" aria-level="8" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="10 11"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;nemeth:number;depth:10" data-semantic-id="10" data-semantic-parent="12" data-semantic-operator="appl" aria-level="9" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c210E">ℎ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.413em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:10" data-semantic-id="11" data-semantic-parent="12" aria-level="9" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="29" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="appl" aria-level="8" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-annotation="depth:9" data-semantic-id="28" data-semantic-children="24,27" data-semantic-parent="30" data-semantic-owns="24 27" aria-level="8" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="24 27"><mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-annotation="depth:10" data-semantic-id="24" data-semantic-children="21" data-semantic-content="22,23" data-semantic-parent="28" data-semantic-owns="22 21 23" aria-level="9" aria-posinset="1" aria-setsize="2" role="treeitem" aria-owns="22 21 23"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="fenced" aria-level="10" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-annotation="depth:11" data-semantic-id="21" data-semantic-children="13,20" data-semantic-content="14" data-semantic-parent="24" data-semantic-owns="13 14 20" aria-level="10" aria-posinset="2" aria-setsize="3" role="treeitem" aria-owns="13 14 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:12" data-semantic-id="13" data-semantic-parent="21" aria-level="11" aria-posinset="1" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-parent="21" data-semantic-operator="infixop,−" aria-level="11" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit;depth:12" data-semantic-id="20" data-semantic-children="17,18" data-semantic-content="19" data-semantic-parent="21" data-semantic-owns="17 19 18" aria-level="11" aria-posinset="3" aria-setsize="3" role="treeitem" aria-owns="17 19 18"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-annotation="depth:13" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="20" data-semantic-owns="15 16" aria-level="12" aria-posinset="1" aria-setsize="3" role="treeitem" aria-owns="15 16"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:14" data-semantic-id="15" data-semantic-parent="17" aria-level="13" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:14" data-semantic-id="16" data-semantic-parent="17" aria-level="13" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢" aria-level="12" aria-posinset="2" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple;depth:13" data-semantic-id="18" data-semantic-parent="20" aria-level="12" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c1D44A">𝑊</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="fenced" aria-level="10" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple;depth:10" data-semantic-id="27" data-semantic-children="26" data-semantic-content="25" data-semantic-parent="28" data-semantic-owns="25 26" aria-level="9" aria-posinset="2" aria-setsize="2" role="treeitem" aria-owns="25 26"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="25" data-semantic-parent="27" data-semantic-operator="prefixop,−" aria-level="10" aria-posinset="1" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple;depth:11" data-semantic-id="26" data-semantic-parent="27" aria-level="10" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="35" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="fenced" aria-level="5" aria-posinset="3" aria-setsize="3" role="treeitem"><mjx-c class="mjx-c29 MM-S3">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-annotation="depth:2" data-semantic-id="37" data-semantic-parent="44" data-semantic-operator="punctuated" aria-level="1" aria-posinset="2" aria-setsize="2" role="treeitem"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo></mjx-math></mjx-container></td>
<td class="label">(4)</td>
</tr></tbody></table>
<p>where <em>α</em>
<sub>
<em>s</em>
</sub> is a parameter that controls spatial dependence (<em>α</em>
<sub>
<em>s</em>
</sub> = 0 implies spatial independence and <em>α</em>
<sub>
<em>s</em>
</sub> = 1 collapses to an intrinsic conditional autoregressive (IAR) specification). <em>D</em> = diag (<em>m</em>
<sub>
<em>i</em>
</sub>) is an <em>n</em> by <em>n</em> diagonal matrix with <em>m</em>
<sub>
<em>i</em>
</sub> = the number of neighbors for pixel <em>i</em>. <em>W</em> is the adjacency matrix (<em>w</em>
<sub>
<em>ii</em>
</sub> = 0, <em>w</em>
<sub>
<em>ij</em>
</sub> = 1 if <em>i</em> is a neighbor of <em>j</em>, and <em>w</em>
<sub>
<em>ij</em>
</sub> = 0 otherwise.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9181802_fgene-13-893522-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5df8/9181802/5a19075ebf3b/fgene-13-893522-g001.jpg" loading="lazy" height="1050" width="749" alt="FIGURE 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of SpatialMap. <strong>(A)</strong> Pipeline of SpatialMap. SpatialMap takes tissue-paired image-based spatial data (contain about 1,000 genes) and scRNA-seq data (contain about 20,000 genes) as inputs, aiming to predict expression patterns for spatially unmeasured genes using a generalized linear spatial model. <strong>(B)</strong> Pearson correlations between predictions and original values of randomly selected 100 genes from experiments on the SeqFISH dataset. The mean values are 0.402, 0.168, and 0.108; minimum values are 0.074, −0.178, and −0.078; and maximum values are 0.878, 0.560, and 0.583 for SpatialMap, SpaGE, and Seurat, respectively. Each dot represents an individual gene. <strong>(C)</strong> Pearson correlations between predictions and original values of SPARK-selected 35 genes from experiments on the SeqFISH dataset. The mean values are 0.607, 0.377, and 0.355; minimum values are 0.024, −0.079, and −0.158; and maximum values are 0.899, 0.764, and 0.899 for SpatialMap, SpaGE, and Seurat, respectively. <strong>(D)</strong> Visualization of predicted spatial expressions. Pearson correlations are 0.825, 0.814, and 0.894 for SpatialMap-performed genes <em>Shank1</em>, <em>Agap2</em>, and <em>Eef1a2</em>, respectively. Color represents the relative gene-expression level (purple, high; green, low) <strong>(E)</strong> Pearson correlations between predictions and original values using different kernel functions in SpatialMap. “CAR 1” to “CAR 9” indicates CAR prior concerning spatial parameters being 0.1 to 0.9, and “Sigma 1” to “Sigma 5” indicates the Gaussian kernel with five chosen various scaling parameters. Color represents different kernels used in SpatialMap.</p></figcaption></figure><p>With the tissue-paired scRNA-seq data and spatial image-based data, we denote <em>x</em>
<sub>
<em>i</em>
</sub> as the average gene expression level across all cells within one particular cell type. The construction of generalized linear spatial models is regarded as the most crucial step. On the one hand, directly modeling on counts avoids information loss caused by the data normalization and scaling process. On the other hand, introducing random effects not only accommodate the widespread independent sequencing error but also explain nonindependent error among pixels due to the affinity in the physical space, fully carrying over-dispersion of sparse spatial transcriptome. Through parameter estimation and statistical inference, the spatial distribution of genes shared by scRNA-seq and spatial techniques can be figured out and served as references for predictions of genes without spatial resolution.</p>
<p>Cross-validation is used to display the power of SpatialMap in reproducing spatially expressed patterns. Specifically, genes shared by tissue-paired spatial and scRNA-seq datasets are used as ground truth. We remove one gene at a time and set the rest as the observed data to fit the generalized linear spatial model (GLSM). Here, for each removed gene, we choose its most correlated gene among the shared data, computed by their scRNA-seq expression profiles, as the reference gene, to predict the spatial patterns of the removed gene using estimates. Finally, correlations between predicted values and their ground truth are used for performance evaluation. We also perform prediction evaluation under different downsampling setups and kernel functions.</p></section><section id="s2-2"><h3 class="pmc_sec_title">2.2 SpatialMap Outperforms SpaGE and Seurat on STARmap and seqFISH Datasets</h3>
<p>We first applied SpatialMap on the tissue-paired seqFISH datasets (<a href="#B15" class="usa-link" aria-describedby="B15" aria-expanded="false">Linus Eng et al., 2019</a>). There were 9,879 shared genes measured by both spatial and single-cell datasets, collected at 913 spatial pixels by the seqFISH technique in the cortex, subventricular zone, and choroid plexus of the mouse brain. For convenience, we only leverage 1,000 randomly selected genes for simulation experiments. We aim to attest excellent prediction performance of SpatialMap, compared with state-of-the-art methods, SpaGE and Seurat. We randomly select 100 genes and apply SpatialMap to the rest of the genes to figure out their spatial distribution for further prediction. SpaGE and Seurat are also performed based on the same settings, respectively. We evaluated the prediction performance by calculating the Pearson correlation between the originally measured spatially resolved genes and their corresponding prediction values. SpatialMap outperforms SpaGE and Seurat with a mean Pearson correlation at 0.388 (average at 0.388 with a standard error at 0.023), compared to a mean at 0.167 and 0.108 for SpaGE and Seurat, respectively (<a href="#F1" class="usa-link">Figure 1B</a>). Moreover, the best prediction for the <em>Syngap1</em> gene reaches a Pearson correlation at 0.878, using SpatialMap, and the highest correlations are 0.560 (<em>Sema6b</em>) and 0.582 (<em>Slain1</em>) (for SpaGE and Seurat, respectively).</p>
<p>We wonder whether having more measured spatial genes is always beneficial to predict the spatial patterns of unmeasured genes. To answer that, we take the number of known genes into account as sensitivity analysis. We fixed 100 genes as a test set and downsampled the remaining 100, 200, 300, 400, 500, and 600 genes to fit the model. However, the average Pearson correlations show only a slight change in prediction performance with a maximum at 0.3973 and a minimum at 0.3963. We check average correlations between the test set and their reference genes under different downsampling setups, and the results show similar minor changes, as the minimum average correlation is 0.1962 and the maximum average correlation is 0.2549, increasing with the number of known genes. Therefore, we may suspect that prediction performance is related to the correlation between unmeasured genes and their reference genes, rather than the number of known genes to fit the model.</p>
<p>Furthermore, since we are concerned with spatial gene-expression patterns, we test the prediction performance for spatially specific genes. SPARK is a method for detecting genes with spatial expression patterns in spatially resolved transcriptomics studies (<a href="#B28" class="usa-link" aria-describedby="B28" aria-expanded="false">Sun et al., 2020</a>). SPARK directly models the count data generated from various spatially resolved transcriptomics techniques through generalized spatial linear models. We apply SPARK on the seqFISH dataset and select 35 significant (adjusted <em>p</em> − value<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf1">
<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="linebreak" tabindex="0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="21" style="font-size: 103.8%;"><mjx-math id="m5" data-semantic-structure="0" class="MM-N"><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-annotation="nemeth:number;depth:1" data-semantic-id="0" aria-level="0" aria-posinset="1" aria-setsize="1" role="treeitem"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo></mjx-math></mjx-container>
</span>0.05) spatially expressed genes. Prediction and correlation evaluation is performed, and the results indicate higher reproducibility of the spatially expressed genes (<a href="#F1" class="usa-link">Figure 1C</a>). The average Pearson correlation achieves 0.607, and the best result even reaches 0.898 (<em>Cnp</em>). We visually compared some highly correlated genes, <em>Shank1</em>, <em>Agap2</em>, and <em>Eef1a2</em> (<a href="#F1" class="usa-link">Figure 1D</a>). SpatialMap recovers their spatial patterns well, while SpaGE and Seurat failed to do so. They mistakenly predicted <em>Shank1</em> to be highly expressed across the tissue, while <em>Agap2</em> remains sparse everywhere.</p>
<p>In addition, we demonstrate the performance of SpatialMap about different kernels with parameters. The Gaussian kernel function with various scaling parameters and CAR prior with different spatial parameters are tested based on 100 genes (<a href="#F1" class="usa-link">Figure 1E</a>). Pearson correlations indicate that SpatialMap is stable to sort kernel functions and shows robustness in parameter selections. We next applied SpatialMap to the tissue-paired STARmap dataset (<a href="#B32" class="usa-link" aria-describedby="B32" aria-expanded="false">Wang et al., 2018</a>). There are 994 shared genes, collected at 1,549 spatial pixels by the STARmap technique, in the mouse’s primary visual cortex. At first, we are interested in reproducing the spatial distribution of known cortical layer marker genes and interneurons. A total of 12 genes including seven excitatory neuron markers (<em>Slc17a1</em>, <em>Nov</em>, <em>Cux2</em>, <em>Rorb</em>, <em>Sulf2</em>, <em>Pcp4</em>, and <em>Ctgf</em>) and five inhibitory neuron markers (<em>Gad1</em>, <em>Pvalb</em>, <em>Sst</em>, <em>Npy</em>, and <em>Vip</em>) remained as a test set. We performed prediction on the rest of the 982 genes using SpaGE and Seurat to benchmark the performance of SpatialMap. We evaluated the prediction performance by calculating the Pearson correlation between the originally measured spatially resolved genes and their corresponding prediction values. The results indicate that SpatialMap outperforms SpaGE and Seurat on spatial pattern reproduction, with a mean Pearson correlation at 0.522, compared to 0.388 and 0.437 for SpaGE and Seurat, respectively (<a href="#F2" class="usa-link">Figure 2B</a>). To be more accurate, 100 spatially resolved genes are selected for testing prediction performance (<a href="#F2" class="usa-link">Figure 2C</a>). We performed using SpaGE and Seurat to benchmark SpatialMap; moreover, different kernel functions with various parameters for the covariance matrix in SpatialMap are tested as well. Pearson correlations demonstrate that SpatialMap works generally better than the other two methods and is fairly stable concerning different parameters.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9181802_fgene-13-893522-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5df8/9181802/2a7958feeed9/fgene-13-893522-g002.jpg" loading="lazy" height="1145" width="748" alt="FIGURE 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance evaluation on STARmap and osmFISH datasets. <strong>(A)</strong> Visualization of predicted spatial expressions. Pearson correlations are 0.320, 0.295, 0.518, 0.736, and 0.564 for SpatialMap-performed genes <em>Nov</em>, <em>Rorb</em>, <em>Pcp4</em>, <em>Gad1</em>, and <em>Vip</em>, respectively. <strong>(B)</strong> Pearson correlations between predictions and original values of 11 known marker genes from experiments on the STARmap dataset. The mean values are 0.522, 0.388, and 0.438; minimum values are 0.295, 0.177, and 0.158; and maximum values are 0.736, 0.565, and 0.650 for SpatialMap, SpaGE, and Seurat, respectively. <strong>(C)</strong> Pearson correlations between predictions and original values of randomly selected 100 genes from experiments on the STARmap dataset. The mean values are 0.250 (average at 0.194 with a standard error at 0.0235 among 10 repeat predictions), 0.142, and 0.129; minimum values are 0.0005, −0.172, and −0.144; and maximum values are 0.734, 0.646, and 0.582 for SpatialMap, SpaGE, and Seurat, respectively. <strong>(D–G)</strong> UMAP visualization of clustering performance on the osmFISH dataset. <strong>(D,E)</strong> Leiden clustering and UMAP based on original 33 genes from the osmFISH dataset, colored by cluster number <strong>(D)</strong> and cell type annotation labels <strong>(E)</strong>. <strong>(F,G)</strong> Leiden clustering and UMAP based on 43 genes by predictions using SpatialMap, colored by cluster number <strong>(F)</strong> and cell type annotation labels <strong>(G)</strong>. Dots represent individual spatial pixels.</p></figcaption></figure><p>We visualize spatially expressed patterns of predictions together with the original truth value for more intuition (<a href="#F2" class="usa-link">Figure 2A</a>). Only part of the genes, <em>Nov</em>, <em>Rorb</em>, <em>Pcp4</em>, <em>Gad1</em>, and <em>Vip</em> are visualized, with the Pearson correlations at 0.328, 0.291, 0.514, 0.719, and 0.575, respectively. Notably, <em>Gad1</em> and <em>Vip</em>, known markers of inhibitory neurons, displayed sparse spatial patterns but showed fairly high reductions using SpatialMap. Therefore, it may indicate the compatibility of SpatialMap to the sparse spatial transcriptome. As can be seen from the aforementioned simulations, although correlations between SpatialMap-predicted values and the original truth are low, it has quite improved power over SpaGE and Seurat and is sufficient to clearly show spatial patterns.</p></section><section id="s2-3"><h3 class="pmc_sec_title">2.3 SpatialMap Accurately Predicts Unmeasured Gene Expression Profiles of OsmFISH and MERFISH Datasets</h3>
<p>The OsmFISH (<a href="#B7" class="usa-link" aria-describedby="B7" aria-expanded="false">Codeluppi et al., 2018</a>) dataset consists of only 33 marker genes of mouse somatosensory cortex tissue by the cyclic smFISH technique. The mouse somatosensory cortex was then divided into 11 specific regions by identifying the spatial-dominant cell types on a single section in previous studies (<a href="#B7" class="usa-link" aria-describedby="B7" aria-expanded="false">Codeluppi et al., 2018</a>). Cross-validation on the OsmFISH dataset showing poor performance is one of the critical reasons for the missing association that is attributed to the small sample size (<a href="#B31" class="usa-link" aria-describedby="B31" aria-expanded="false">Wang et al., 2022</a>). In the original dataset, Leiden clustering was performed on all 33 gene-expression profiles. However, visualization by UMAP (performed using five dimensions) does not clearly show distinct clusters (<a href="#F2" class="usa-link">Figures 2D,E</a>). Therefore, we consider improving the clustering based on predicting more spatially resolved gene patterns. After performing SpatialMap, UMAP visualization showed clear clusters (<a href="#F2" class="usa-link">Figures 2F,G</a>), consistent with the fact that two clusters far apart are also anatomically separate. Therefore, by predicting the spatial expression profiles, it is helpful to optimize the spatial clustering, which is conducive to the division of a data-driven spatial organization.</p>
<p>Next, based on the previous simulation experiments, SpatialMap has shown great predictive power in leave-<em>n</em>-genes-out cross-validations. However, all of these results were achieved using relatively small-scale spatial datasets, with SeqFISH, STARmap, and OsmFISH measured on just around 1,000 pixels. This raises the question of whether SpatialMap is capable of applications on larger-scale spatial datasets. To evaluate this performance, the MERFISH (<a href="#B33" class="usa-link" aria-describedby="B33" aria-expanded="false">Xia et al., 2019</a>) spatial dataset together with its corresponding scRNA-seq dataset pair, collected from the preoptic region of the thalamus in mice, was used for simulations. The MERFISH dataset contains transcriptomics profiles of 160 genes measured on 4,975 pixels. Of these, 155 genes were selected in previous studies as the markers of different cell populations or associated with various neuronal functions. A total of 139 genes were shared, and four marker genes were selected for four main cell types, inhibitory neuron (<em>Gad1</em>), mature OD cells (<em>Mbp</em>), ependymal cells (<em>Cd24a</em>), and mural cells (<em>Myh11</em>). We apply SpatialMap to reproduce their spatial patterns and perform visualization compared with SpaGE and Seurat.</p>
<p>The results indicated that SpatialMap could reproduce their spatially expressed patterns (<a href="#F3" class="usa-link">Figure 3B</a>) and is consistent with the distribution of cell types (<a href="#F3" class="usa-link">Figure 3B</a>). Compared with SpaGE and Seurat, SpatialMap shows clear spatial expression patterns and avoids overexpressed predictions, especially for the <em>Gad1</em> marker, while SpaGE and Seurat predicted a highly expressed pattern in the truly low-expressed regions. However, for the marker gene <em>Cd24a</em>, SpatialMap is equally unsatisfactory. It may be because of the low correlation (0.179) between the <em>Cd24a</em>-expression level and any other reference gene. Therefore, genes with distinct differences at scRNA-seq levels are not suitable as references for spatial pattern predictions.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9181802_fgene-13-893522-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5df8/9181802/a1ca4959321e/fgene-13-893522-g003.jpg" loading="lazy" height="515" width="748" alt="FIGURE 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SpatialMap accurately predicted the expression of marker genes <em>Gad1</em>, <em>Mbp</em>, <em>Myhh11</em>, and <em>Cd24a</em>, consistent with MERFISH-measured data. <strong>(A)</strong> Spatial distribution of four main cell classes. The cell classes are represented by colored dots, while all other background cells are shown as gray dots. <strong>(B)</strong> SpatialMap-predicted genes (the second column) fairly recover their measured patterns by the MERFISH technique (the first column) and correspond to the distribution of cell types. Color represents the relative gene-expression level (purple, high; green, low).</p></figcaption></figure></section></section><section id="s3"><h2 class="pmc_sec_title" data-anchor-id="s3"><a id="s3-anchor" data-anchor-id="s3" class="usa-anchor"></a>3 Methods and Materials</h2>
<section id="s3-1"><h3 class="pmc_sec_title">3.1 SpatialMap: Model and Algorithm</h3>
<p>We aim to model gene-expression data collected by various image-based spatial technologies, such as seqFISH, osmFISH, and MERFISH, using integration with single-cell RNA sequencing data referred to the same biological tissue. These spatial techniques simultaneously record gene-expression levels of <em>q</em> different genes on <em>n</em> different spatial sites on a tissue of interest, which we also refer to as spatial pixels. The gene-expression levels are usually obtained as counts for the fact that they are numbers of barcode mRNA for any given transcript in a single cell. The number of genes, <em>q</em>, often varies as many as ∼1,000, and the number of spatial sites randomly varies from a couple of hundreds to thousands. For the corresponding scRNA-seq data, the number of genes can achieve the whole transcriptome for every single cell.</p>
<p>The purpose to integrate image-based spatially resolved data with scRNA-seq collected on the same tissue is to predict spatially expressed patterns for those unmeasured genes. We map single-cell level expression patterns for cell types using a generalized linear spatial model (GLSM). GLSM is a generalized linear mixed model that directly models non-Gaussian spatial data and uses random effects to capture the underlying stationary spatial process (<a href="#B6" class="usa-link" aria-describedby="B6" aria-expanded="false">Christensen and Waagepetersen, 2002</a>).</p>
<p>Specifically, SpatialMap uses two gene-expression matrices <em>Y</em>
<sub>
<em>n</em>×<em>q</em>
</sub> and <em>S</em>
<sub>
<em>m</em>×<em>p</em>
</sub> as inputs, corresponding to image-based spatial transcriptome data and scRNA-seq, respectively. We model one gene at a time. We denote <em>y</em>
<sub>
<em>i</em>
</sub>, the <em>i</em>th row of <em>Y</em>
<sub>
<em>n</em>×<em>q</em>
</sub>, as the gene-expression measurement in the form of counts for the <em>i</em>th spatial pixel. We denote <em>N</em>
<sub>
<em>i</em>
</sub> as the normalization factor for the <em>i</em>th pixel, computed as the summation of the total number of counts across all genes for the <em>i</em>th pixel. We consider modeling the observed <em>y</em>
<sub>
<em>i</em>
</sub> with an overdispersed Poisson distribution:</p>
<table class="disp-formula p" id="e5"><tbody><tr>
<td class="formula"><mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="5"><mjx-lazy data-mjx-lazy="5"></mjx-lazy></mjx-container></td>
<td class="label">(5)</td>
</tr></tbody></table>
<p>where <em>Poi</em> (⋅) is the Poisson distribution and <em>λ</em>
<sub>
<em>i</em>
</sub> is an unknown Poisson rate parameter, which can be modeled as the linear combination of the following terms:</p>
<table class="disp-formula p" id="e6"><tbody><tr>
<td class="formula"><mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="6"><mjx-lazy data-mjx-lazy="6"></mjx-lazy></mjx-container></td>
<td class="label">(6)</td>
</tr></tbody></table>
<table class="disp-formula p" id="e7"><tbody><tr>
<td class="formula"><mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="7"><mjx-lazy data-mjx-lazy="7"></mjx-lazy></mjx-container></td>
<td class="label">(7)</td>
</tr></tbody></table>
<table class="disp-formula p" id="e8"><tbody><tr>
<td class="formula"><mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="8"><mjx-lazy data-mjx-lazy="8"></mjx-lazy></mjx-container></td>
<td class="label">(8)</td>
</tr></tbody></table>
<p>where <em>α</em> is an intercept; <em>x</em>
<sub>
<em>i</em>
</sub> refers to the expression-level of the cell type mapped to the <em>i</em>th pixel; <em>β</em> is the corresponding coefficient; <em>e</em>
<sub>
<em>i</em>
</sub> is the residual error that is independently and identically distributed following <em>N</em> (0, <em>σ</em>
<sup>2</sup> (1 − <em>h</em>
<sup>2</sup>)), with <em>σ</em>
<sub>2</sub> and <em>h</em>
<sup>2</sup> being the scaling factors; and <em>g</em>
<sub>
<em>i</em>
</sub> is a random effect modeling the spatial correlation pattern among spatial pixels, where <em>MVN</em> (⋅) denotes the multivariate normal distribution and the variance–covariance matrix and <em>K</em> is a kernel function of the <em>n</em> spatial pixels using Gaussian kernel or CAR prior. We set <em>h</em>
<sup>2</sup> ∈ [0, 1], explaining the expression variance in log (<em>λ</em>
<sub>
<em>i</em>
</sub>) owing to the random noise <em>g</em>. <em>e</em> follows a multivariate normal distribution, and <em>I</em> is a <em>n</em> by <em>n</em> identity matrix.</p>
<p>For image-based spatial data, each count represents a gene-expression measurement in a single cell at that pixel. Therefore, finding an embedded mapping between single cells and spatial pixels can directly make the integration work. However, due to inconsistency in the number of cells and number of pixels, mapping between cell types and spatial pixels is constructed as a replacement due to the following considerations: first, tissue dissociation before scRNA-seq is unlikely to preserve all the cells, resulting in information imbalance between single-cell and spatial techniques. In addition, spatial data and scRNA-seq data are often collected from two different experiments, hardly retaining one-to-one correspondence among pixels and cells. Moreover, the spatial characteristics of transcriptomes and their spontaneous impacts are the research purpose instead of the precise physical location of each cell in a tissue. Therefore, we set the gene-expression level for each cell type as the average of cells within that type, simply referred to as the center. The center is mapped to its corresponding pixels based on Pearson correlation in the expression space. Cell type identification can be based on previous annotations or other clustering methods.</p>
<p>We evaluated the prediction performance using cross-validation. <em>N</em> genes shared by spatial and scRNA-seq datasets are used as ground truth; <em>n</em> genes are left out, and the remaining <em>N</em> − <em>n</em> genes are used for integration and spatial mapping of <em>n</em> “unmeasured” expression profiles in spatial data. The prediction is then evaluated by comparing the original measurement and the predicted expression profiles of the removed genes. We have two kinds of gene left-out experiments. On the one hand, we intend to randomly select some genes and evaluate the prediction performance of all the methods to draw their ability in general. On the other hand, considering that we are interested in genes that display spatial patterns rather than their exact gene expression level, we identify some genes with spatial expression patterns (referred to as SE genes) and perform prediction evaluations, to characterize the performance of all the methods on them.</p></section><section id="s3-2"><h3 class="pmc_sec_title">3.2 Algorithms</h3>
<p>In GLSM, whether the single-cell gene-expression level that is helpful to characterize its spatial pattern conducive to the selected genes which serve well as a reference to predict spatial patterns for more unmeasured ones is tested. Also, it can be translated into testing the null hypothesis “<em>H</em>
<sub>0</sub>: <em>β</em> = 0.” Parameter estimation and hypothesis testing in GLSM is notoriously difficult, as the GLSM likelihood consists of an <em>n</em>-dimensional integral that cannot be solved analytically. Therefore, we develop an approximate-inference algorithm based on the penalized quasi-likelihood (PQL) approach (<a href="#B13" class="usa-link" aria-describedby="B13" aria-expanded="false">Lin and Norman, 1996</a>; <a href="#B27" class="usa-link" aria-describedby="B27" aria-expanded="false">Sun et al., 2019</a>). The PQL approach employs an iterative numerical optimization procedure. In each iteration, we introduce a set of continuous pseudodata <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf2">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="9"><mjx-lazy data-mjx-lazy="9"></mjx-lazy></mjx-container>
</span> to replace the originally observed count data <em>y</em>. The pseudodata <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf3">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="10"><mjx-lazy data-mjx-lazy="10"></mjx-lazy></mjx-container>
</span> is obtained based on a second-order Taylor expansion using the conditional distribution <em>P</em> (<em>y</em>
<sub>
<em>i</em>
</sub>|<em>β</em>, <em>e</em>) with the first- and second-order moments <em>E</em> (<em>y</em>
<sub>
<em>i</em>
</sub>|<em>β</em>, <em>e</em>) and <em>V</em> (<em>y</em>
<sub>
<em>i</em>
</sub>|<em>β</em>, <em>e</em>), both evaluated at the current estimates of the fixed coefficients <em>β</em> as well as the nugget effects <em>e</em>. With the pseudodata, the complex GLSM likelihood function for the original data <em>y</em> is replaced by a much simpler LMM likelihood function for the pseudodata <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf4">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="11"><mjx-lazy data-mjx-lazy="11"></mjx-lazy></mjx-container>
</span>, thereby alleviating much of the computational burden associated with GLSM. With the pseudodata <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf5">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="12"><mjx-lazy data-mjx-lazy="12"></mjx-lazy></mjx-container>
</span>, we can perform inference and update parameters using the standard average information (AI) algorithm for LMMs (<a href="#B34" class="usa-link" aria-describedby="B34" aria-expanded="false">Yang et al., 2011</a>; <a href="#B9" class="usa-link" aria-describedby="B9" aria-expanded="false">Han et al., 2016</a>). By iterating between the approximation step of obtaining the pseudodata <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf6">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="13"><mjx-lazy data-mjx-lazy="13"></mjx-lazy></mjx-container>
</span> and the inference step of updating the parameter estimates <em>via</em> the AI algorithm, the PQL approach allows us to perform inference in a computationally efficient fashion. To further improve the computational speed, we also take advantage of the parallel computing environment readily available in modern desktop computers nowadays and implement our method with multiple-thread computing capability using Rcpp.</p>
<p>The statistical power of such a hypothesis test will inevitably depend on how well the spatial kernel function <em>K</em> matches the true underlying spatial patterns. Therefore, the Gaussian kernel with different scaling parameters and kernel from the conditional auto-regression (CAR) model (<a href="#B8" class="usa-link" aria-describedby="B8" aria-expanded="false">Duncan et al., 2014</a>) with different spatial parameters are performed in experiments to show quite a robustness of SpatialMap. With parameter estimates from the PQL-based approach, we compute the <em>p</em> − value and select genes with <em>p</em> − value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf7">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="14"><mjx-lazy data-mjx-lazy="14"></mjx-lazy></mjx-container>
</span> as a reference set. For those spatially unmeasured genes, we compute their Pearson correlation with the reference set in the single-cell expression space and extract the most “correlated” gene with its estimates <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf8">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="15"><mjx-lazy data-mjx-lazy="15"></mjx-lazy></mjx-container>
</span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="inf9">
<mjx-container tabindex="0" class="CtxtMenu_Attached_0" ctxtmenu_oldtabindex="0" ctxtmenu_counter="16"><mjx-lazy data-mjx-lazy="16"></mjx-lazy></mjx-container>
</span> is computed with <em>X</em>
<sup>(<em>un</em>)</sup> being valued of that spatially unmeasured gene, and its predicted spatial pattern can be finally sampled.</p></section><section id="s3-3"><h3 class="pmc_sec_title">3.3 Datasets Used in This Study</h3>
<p>We used four image-based spatial datasets (SeqFISH (<a href="#B15" class="usa-link" aria-describedby="B15" aria-expanded="false">Linus Eng et al., 2019</a>), STARmap (<a href="#B32" class="usa-link" aria-describedby="B32" aria-expanded="false">Wang et al., 2018</a>), OsmFISH (<a href="#B7" class="usa-link" aria-describedby="B7" aria-expanded="false">Codeluppi et al., 2018</a>), and MERFISH (<a href="#B33" class="usa-link" aria-describedby="B33" aria-expanded="false">Xia et al., 2019</a>)) and two scRNA-seq datasets (AllenVISp (<a href="#B29" class="usa-link" aria-describedby="B29" aria-expanded="false">Tasic et al., 2018</a>) and Moffit (<a href="#B19" class="usa-link" aria-describedby="B19" aria-expanded="false">Moffitt Jeffrey et al., 2018</a>)). For image-based data, the SeqFISH dataset contains 10,000-gene probes to image 913 cells in the cortex, subventricular zone (SVZ), and choroid plexus of the mouse brain using the SeqFISH+ technique. It can be downloaded at <a href="https://github.com/CaiGroup/seqFISH-PLUS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/CaiGroup/seqFISH-PLUS</a>. The STARmap dataset contains 1,020 genes on 1,549 cells using the STARmap technique of the mouse primary visual cortex. The corresponding data and preprocessing code can be downloaded at <a href="https://github.com/weallen/STARmap" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/weallen/STARmap</a>. The OsmFISH dataset contains 33 genes of 1,290 cells collected from the mouse somatosensory cortex using cyclic smFISH. It can be downloaded at <a href="http://linnarssonlab.org/osmFISH/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://linnarssonlab.org/osmFISH/</a>. The MERFISH dataset is collected on the mouse preoptic region of the hypothalamus from Dryad. We used the slice at Bregma +0.11&nbsp;mm from animal 18 for analysis, as it contains 155 genes measured on 4,975 cells. It can be downloaded at <a href="https://datadryad.org/stash/dataset/doi:10.5061/dryad.8t8s248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://datadryad.org/stash/dataset/doi:10.5061/dryad.8t8s248</a>. The downloaded data have already been normalized. To obtain the raw count data, we rescaled the expression values by first multiplying by 1,000, adjusting for cell volume, and then converting the rescaled value into integers by taking the ceiling over the rescaled data. For scRNA-seq data, the AllenVISp (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115746" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE115746</a>) dataset contains scRNA-seq of 12,714 cells from the mouse primary visual cortex (VISp) with cell-type annotations. The Moffit (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113576" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE113576</a>) dataset contains scRNA-seq of ∼ 31,000 cells dissociated and captured from the preoptic region of the hypothalamus from multiple male and female mice.</p>
<p>During preprocessing of AllenVISp data, cells with the read depth (computed by the summation of the read counts across genes) less than 10 were removed, and cells labeled as “No class” or “Low Quality” are also left out according to the known metadata. During preprocessing of Moffit data, cells with the read depth less than 100 or labeled as “Ambiguous” or “Unstable” were removed. Cell filtration keeps the same for all the three methods. SpaGE and Seurat require data normalization (log-transformation using scaling factor 1e4), while SpatialMap takes raw count data as input. Other details on preprocessing of four spatial datasets are described in the Results part.</p></section></section><section id="s4"><h2 class="pmc_sec_title" data-anchor-id="s4"><a id="s4-anchor" data-anchor-id="s4" class="usa-anchor"></a>4 Discussion</h2>
<p>We develop a computational method, SpatialMap, based on the generalized linear mixed model for mapping scRNA-seq into the spatial landscape. The predictive performance of SpatialMap is highly dependent on the construction and inference of statistical models for spatial gene expression patterns. In a statistical sense, the spatial distribution characteristics of each gene are outlined more directly and clearly, providing a reliable distribution rule for the prediction of the spatial pattern of new genes. In the process of data preprocessing, SpatialMap only needs to filter the pixels or genes with low expression and does not need to normalize or scale the data, to greatly avoid the interference and ambiguity of the latent information during the classical data preprocessing.</p>
<p>By simulations, we demonstrate that SpatialMap can effectively restore the spatial patterns of genes with significant spatial expression patterns. Compared with the present methods, SpatialMap can better display the relative gradient of gene distribution. For extremely sparse genes, SpatialMap can also capture their distribution to some extent. However, genes with highly sparse measurements in the downstream analysis generally do not provide significant information. Thus, it is recommended to be filtered in the preprocessing step. Comparing the prediction performance in SeqFISH data under the cross-validation set up of the randomly-selected genes and SPARK-selected-SE genes, SpatialMap managed to achieve higher correlations in genes with spatial patterns. We suspect that there are two reasons: on the one hand, SPARK can identify only 35 genes as SE genes, so the high correlations of some individual genes lead to a higher average level; on the other hand, we fit the models using genes with both spatial measurements and scRNA-seq expression profiles, among which the most correlated one in the scRNA-seq expression space is selected as the reference to predict spatially unmeasured genes. The average correlation level between the 100 randomly selected genes and their reference genes in scRNA-seq is 0.206 (with a minimum at 0.068 and a maximum at 0.674), while the average correlation level between the 35 SE genes selected by SPARK and their reference genes in scRNA-seq is 0.350 (with a minimum at 0.022 and a maximum at 0.616). Therefore, measured in the scRNA-seq expression space, the more dissimilar the reference gene is to the gene to be predicted, the worse the predictive performance will be.</p>
<p>SpatialMap can directly capture data characteristics. Based on the relationship between the expression of genes at the single-cell resolution and the configuration of spatial patterns, SpatialMap can fit the spatial distributions of genes well by establishing adaptive distribution and is supported by a powerful parameter estimation algorithm, which greatly improves the application and efficiency of the method.</p></section><section id="s5"><h2 class="pmc_sec_title" data-anchor-id="s5"><a id="s5-anchor" data-anchor-id="s5" class="usa-anchor"></a>Data Availability Statement</h2>
<p>All source code used in our experiments and the processed data has been deposited at <a href="https://sqsun.github.io/software.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sqsun.github.io/software.html</a>.</p></section><section id="s6"><h2 class="pmc_sec_title" data-anchor-id="s6"><a id="s6-anchor" data-anchor-id="s6" class="usa-anchor"></a>Author Contributions</h2>
<p>SS and XD conceived the idea and provided funding support. SS designed the experiments. JN and DG performed the real data analyses. JN, DG, GL, SS, and XD revised the manuscript. JN, DG, SS, and XD wrote the manuscript.</p></section><section id="s7"><h2 class="pmc_sec_title" data-anchor-id="s7"><a id="s7-anchor" data-anchor-id="s7" class="usa-anchor"></a>Funding</h2>
<p>This study was supported by the National Natural Science Foundation-Outstanding Youth Foundation (Grant No. 82122061) and the National Natural Science Foundation of China (Grant No. 61902319).</p></section><section id="s8"><h2 class="pmc_sec_title" data-anchor-id="s8"><a id="s8-anchor" data-anchor-id="s8" class="usa-anchor"></a>Conflict of Interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="s9"><h2 class="pmc_sec_title" data-anchor-id="s9"><a id="s9-anchor" data-anchor-id="s9" class="usa-anchor"></a>Publisher’s Note</h2>
<p>All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors, and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list1"><a id="ref-list1-anchor" data-anchor-id="ref-list1" class="usa-anchor"></a>References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>
Abdelaal T., Mourragui S., Ahmed M., Marcel J., Spage T. R. (2020). Spatial Gene Enhancement Using Scrna-Seq. Nucleic Acids Res.
48 (18), e107. 10.1093/nar/gkaa740
</cite> [<a href="https://doi.org/10.1093/nar/gkaa740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7544237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32955565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Spatial%20Gene%20Enhancement%20Using%20Scrna-Seq&amp;author=T.%20Abdelaal&amp;author=S.%20Mourragui&amp;author=M.%20Ahmed&amp;author=J.%20Marcel&amp;author=T.%20R.%20Spage&amp;volume=48&amp;issue=18&amp;publication_year=2020&amp;pages=e107&amp;pmid=32955565&amp;doi=10.1093/nar/gkaa740&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<cite>
Achim K., Pettit J.-B., Saraiva L. R., Gavriouchkina D., Larsson T., Arendt D., et al.  (2015). High-throughput Spatial Mapping of Single-Cell Rna-Seq Data to Tissue of Origin. Nat. Biotechnol.
33 (5), 503–509. 10.1038/nbt.3209
</cite> [<a href="https://doi.org/10.1038/nbt.3209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25867922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=High-throughput%20Spatial%20Mapping%20of%20Single-Cell%20Rna-Seq%20Data%20to%20Tissue%20of%20Origin&amp;author=K.%20Achim&amp;author=J.-B.%20Pettit&amp;author=L.%20R.%20Saraiva&amp;author=D.%20Gavriouchkina&amp;author=T.%20Larsson&amp;volume=33&amp;issue=5&amp;publication_year=2015&amp;pages=503-509&amp;pmid=25867922&amp;doi=10.1038/nbt.3209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<cite>
Antebi Y. E., Linton J. M., Klumpe H., Bintu B., Gong M., Su C., et al.  (2017). Combinatorial Signal Perception in the Bmp Pathway. Cell.
170 (6), 1184–1196. 10.1016/j.cell.2017.08.015
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5612783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28886385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.&amp;title=Combinatorial%20Signal%20Perception%20in%20the%20Bmp%20Pathway&amp;author=Y.%20E.%20Antebi&amp;author=J.%20M.%20Linton&amp;author=H.%20Klumpe&amp;author=B.%20Bintu&amp;author=M.%20Gong&amp;volume=170&amp;issue=6&amp;publication_year=2017&amp;pages=1184-1196&amp;pmid=28886385&amp;doi=10.1016/j.cell.2017.08.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<cite>
Battich N., Stoeger T., Pelkmans L. (2013). Image-based Transcriptomics in Thousands of Single Human Cells at Single-Molecule Resolution. Nat. Methods
10 (11), 1127–1133. 10.1038/nmeth.2657
</cite> [<a href="https://doi.org/10.1038/nmeth.2657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24097269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Image-based%20Transcriptomics%20in%20Thousands%20of%20Single%20Human%20Cells%20at%20Single-Molecule%20Resolution&amp;author=N.%20Battich&amp;author=T.%20Stoeger&amp;author=L.%20Pelkmans&amp;volume=10&amp;issue=11&amp;publication_year=2013&amp;pages=1127-1133&amp;pmid=24097269&amp;doi=10.1038/nmeth.2657&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<cite>
Burgess D. J. (2019). Spatial Transcriptomics Coming of Age. Nat. Rev. Genet.
20 (6), 317. 10.1038/s41576-019-0129-z
</cite> [<a href="https://doi.org/10.1038/s41576-019-0129-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30980030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Spatial%20Transcriptomics%20Coming%20of%20Age&amp;author=D.%20J.%20Burgess&amp;volume=20&amp;issue=6&amp;publication_year=2019&amp;pages=317&amp;pmid=30980030&amp;doi=10.1038/s41576-019-0129-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<cite>
Christensen O. F., Waagepetersen R. (2002). Bayesian Prediction of Spatial Count Data Using Generalized Linear Mixed Models. Biometrics
58 (2), 280–286. 10.1111/j.0006-341x.2002.00280.x
</cite> [<a href="https://doi.org/10.1111/j.0006-341x.2002.00280.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12071400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Bayesian%20Prediction%20of%20Spatial%20Count%20Data%20Using%20Generalized%20Linear%20Mixed%20Models&amp;author=O.%20F.%20Christensen&amp;author=R.%20Waagepetersen&amp;volume=58&amp;issue=2&amp;publication_year=2002&amp;pages=280-286&amp;pmid=12071400&amp;doi=10.1111/j.0006-341x.2002.00280.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<cite>
Codeluppi S., Borm L. E., Amit Z., La Manno G., Josina A., van Lunteren J. A., et al.  (2018). Spatial Organization of the Somatosensory Cortex Revealed by Osmfish. Nat. Methods
15 (11), 932–935. 10.1038/s41592-018-0175-z
</cite> [<a href="https://doi.org/10.1038/s41592-018-0175-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30377364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Spatial%20Organization%20of%20the%20Somatosensory%20Cortex%20Revealed%20by%20Osmfish&amp;author=S.%20Codeluppi&amp;author=L.%20E.%20Borm&amp;author=Z.%20Amit&amp;author=G.%20La%20Manno&amp;author=A.%20Josina&amp;volume=15&amp;issue=11&amp;publication_year=2018&amp;pages=932-935&amp;pmid=30377364&amp;doi=10.1038/s41592-018-0175-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<cite>
Duncan L., Rushworth A., Sujit K S. (2014). A Bayesian Localized Conditional Autoregressive Model for Estimating the Health Effects of Air Pollution. Biometrics
70 (2), 419–429. 10.1111/biom.12156
</cite> [<a href="https://doi.org/10.1111/biom.12156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4282098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24571082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=A%20Bayesian%20Localized%20Conditional%20Autoregressive%20Model%20for%20Estimating%20the%20Health%20Effects%20of%20Air%20Pollution&amp;author=L.%20Duncan&amp;author=A.%20Rushworth&amp;author=K%20S.%20Sujit&amp;volume=70&amp;issue=2&amp;publication_year=2014&amp;pages=419-429&amp;pmid=24571082&amp;doi=10.1111/biom.12156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<cite>
Han C., Wang C., Conomos M. P., Stilp A. M., Li Z., Sofer T., et al.  (2016). Control for Population Structure and Relatedness for Binary Traits in Genetic Association Studies via Logistic Mixed Models. Am. J. Hum. Genet.
98 (4), 653–666. 10.1016/j.ajhg.2016.02.012
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2016.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4833218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27018471/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&amp;title=Control%20for%20Population%20Structure%20and%20Relatedness%20for%20Binary%20Traits%20in%20Genetic%20Association%20Studies%20via%20Logistic%20Mixed%20Models&amp;author=C.%20Han&amp;author=C.%20Wang&amp;author=M.%20P.%20Conomos&amp;author=A.%20M.%20Stilp&amp;author=Z.%20Li&amp;volume=98&amp;issue=4&amp;publication_year=2016&amp;pages=653-666&amp;pmid=27018471&amp;doi=10.1016/j.ajhg.2016.02.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<cite>
James E., Sul J.-Y., Bartfai T., Kim J. (2014). The Promise of Single-Cell Sequencing. Nat. Methods
11 (1), 25–27. 10.1038/nmeth.2769
</cite> [<a href="https://doi.org/10.1038/nmeth.2769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24524134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=The%20Promise%20of%20Single-Cell%20Sequencing&amp;author=E.%20James&amp;author=J.-Y.%20Sul&amp;author=T.%20Bartfai&amp;author=J.%20Kim&amp;volume=11&amp;issue=1&amp;publication_year=2014&amp;pages=25-27&amp;pmid=24524134&amp;doi=10.1038/nmeth.2769&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<cite>
Julian B., Kooperberg C. (1995). On Conditional and Intrinsic Autoregression. Biometrika
82 (4), 733–746. 10.2307/2337341
</cite> [<a href="https://doi.org/10.2307/2337341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=On%20Conditional%20and%20Intrinsic%20Autoregression&amp;author=B.%20Julian&amp;author=C.%20Kooperberg&amp;volume=82&amp;issue=4&amp;publication_year=1995&amp;pages=733-746&amp;doi=10.2307/2337341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<cite>
Levy-Jurgenson A., Tekpli X., Kristensen V. N., Zohar Y. (2020). Spatial Transcriptomics Inferred from Pathology Whole-Slide Images Links Tumor Heterogeneity to Survival in Breast and Lung Cancer. Sci. Rep.
10 (1), 18802. 10.1038/s41598-020-75708-z
</cite> [<a href="https://doi.org/10.1038/s41598-020-75708-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7606448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33139755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Spatial%20Transcriptomics%20Inferred%20from%20Pathology%20Whole-Slide%20Images%20Links%20Tumor%20Heterogeneity%20to%20Survival%20in%20Breast%20and%20Lung%20Cancer&amp;author=A.%20Levy-Jurgenson&amp;author=X.%20Tekpli&amp;author=V.%20N.%20Kristensen&amp;author=Y.%20Zohar&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=18802&amp;pmid=33139755&amp;doi=10.1038/s41598-020-75708-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<cite>
Lin X., Norman E. (1996). Breslow. Bias Correction in Generalized Linear Mixed Models with Multiple Components of Dispersion. J. Am. Stat. Assoc.
91 (435), 1007–1016. 10.1080/01621459.1996.10476971
</cite> [<a href="https://doi.org/10.1080/01621459.1996.10476971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Stat.%20Assoc.&amp;title=Breslow.%20Bias%20Correction%20in%20Generalized%20Linear%20Mixed%20Models%20with%20Multiple%20Components%20of%20Dispersion&amp;author=X.%20Lin&amp;author=E.%20Norman&amp;volume=91&amp;issue=435&amp;publication_year=1996&amp;pages=1007-1016&amp;doi=10.1080/01621459.1996.10476971&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<cite>
Lindeboom R. G. H., Regev A., Teichmann S. A. (2021). Towards a Human Cell Atlas: Taking Notes from the Past. Trends Genet.
37 (7), 625–630. 10.1016/j.tig.2021.03.007
</cite> [<a href="https://doi.org/10.1016/j.tig.2021.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33879355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&amp;title=Towards%20a%20Human%20Cell%20Atlas:%20Taking%20Notes%20from%20the%20Past&amp;author=R.%20G.%20H.%20Lindeboom&amp;author=A.%20Regev&amp;author=S.%20A.%20Teichmann&amp;volume=37&amp;issue=7&amp;publication_year=2021&amp;pages=625-630&amp;pmid=33879355&amp;doi=10.1016/j.tig.2021.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<cite>
Linus Eng C.-H., Lawson M., Zhu Q., Dries R., Koulena N., Takei Y., et al.  (2019). Transcriptome-scale Super-resolved Imaging in Tissues by Rna Seqfish+. Nature
568 (7751), 235–239. 10.1038/s41586-019-1049-y
</cite> [<a href="https://doi.org/10.1038/s41586-019-1049-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6544023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30911168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome-scale%20Super-resolved%20Imaging%20in%20Tissues%20by%20Rna%20Seqfish+&amp;author=C.-H.%20Linus%20Eng&amp;author=M.%20Lawson&amp;author=Q.%20Zhu&amp;author=R.%20Dries&amp;author=N.%20Koulena&amp;volume=568&amp;issue=7751&amp;publication_year=2019&amp;pages=235-239&amp;pmid=30911168&amp;doi=10.1038/s41586-019-1049-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<cite>
Lun A. (2018). Overcoming Systematic Errors Caused by Log-Transformation of Normalized Single-Cell Rna Sequencing Data. bioRxiv, 404962. 10.1101/404962
</cite> [<a href="https://doi.org/10.1101/404962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Overcoming%20Systematic%20Errors%20Caused%20by%20Log-Transformation%20of%20Normalized%20Single-Cell%20Rna%20Sequencing%20Data&amp;author=A.%20Lun&amp;publication_year=2018&amp;pages=404962&amp;doi=10.1101/404962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<cite>
Ma Y., Sun S., Shang X., Keller E. T., Chen M., Zhou X. (2020). Integrative Differential Expression and Gene Set Enrichment Analysis Using Summary Statistics for Scrna-Seq Studies. Nat. Commun.
11 (1), 1585. 10.1038/s41467-020-15298-6
</cite> [<a href="https://doi.org/10.1038/s41467-020-15298-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7101316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32221292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Integrative%20Differential%20Expression%20and%20Gene%20Set%20Enrichment%20Analysis%20Using%20Summary%20Statistics%20for%20Scrna-Seq%20Studies&amp;author=Y.%20Ma&amp;author=S.%20Sun&amp;author=X.%20Shang&amp;author=E.%20T.%20Keller&amp;author=M.%20Chen&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=1585&amp;pmid=32221292&amp;doi=10.1038/s41467-020-15298-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<cite>
Mereu E., Lafzi A., Moutinho C., Ziegenhain C., McCarthy D. J., Álvarez-Varela A., et al.  (2020). Benchmarking Single-Cell Rna-Sequencing Protocols for Cell Atlas Projects. Nat. Biotechnol.
38 (6), 747–755. 10.1038/s41587-020-0469-4
</cite> [<a href="https://doi.org/10.1038/s41587-020-0469-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32518403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Benchmarking%20Single-Cell%20Rna-Sequencing%20Protocols%20for%20Cell%20Atlas%20Projects&amp;author=E.%20Mereu&amp;author=A.%20Lafzi&amp;author=C.%20Moutinho&amp;author=C.%20Ziegenhain&amp;author=D.%20J.%20McCarthy&amp;volume=38&amp;issue=6&amp;publication_year=2020&amp;pages=747-755&amp;pmid=32518403&amp;doi=10.1038/s41587-020-0469-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<cite>
Moffitt Jeffrey R., Bambah-Mukku D., Eichhorn Stephen W., Vaughn E., Shekhar K., Perez Julio D., et al.  (2018). Aviv Regev, Catherine Dulac, and Xiaowei Zhuang. Molecular, Spatial, and Functional Single-Cell Profiling of the Hypothalamic Preoptic Region. Science
362 (6416), eaau5324. 10.1126/science.aau5324
</cite> [<a href="https://doi.org/10.1126/science.aau5324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6482113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30385464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Aviv%20Regev,%20Catherine%20Dulac,%20and%20Xiaowei%20Zhuang.%20Molecular,%20Spatial,%20and%20Functional%20Single-Cell%20Profiling%20of%20the%20Hypothalamic%20Preoptic%20Region&amp;author=R.%20Moffitt%20Jeffrey&amp;author=D.%20Bambah-Mukku&amp;author=W.%20Eichhorn%20Stephen&amp;author=E.%20Vaughn&amp;author=K.%20Shekhar&amp;volume=362&amp;issue=6416&amp;publication_year=2018&amp;pages=eaau5324&amp;pmid=30385464&amp;doi=10.1126/science.aau5324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<cite>
Picelli S., Faridani O. R., Björklund A. K., Winberg G., Sagasser S., Sandberg R. (2014). Full-length Rna-Seq from Single Cells Using Smart-Seq2. Nat. Protoc.
9 (1), 171–181. 10.1038/nprot.2014.006
</cite> [<a href="https://doi.org/10.1038/nprot.2014.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24385147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Full-length%20Rna-Seq%20from%20Single%20Cells%20Using%20Smart-Seq2&amp;author=S.%20Picelli&amp;author=O.%20R.%20Faridani&amp;author=A.%20K.%20Bj%C3%B6rklund&amp;author=G.%20Winberg&amp;author=S.%20Sagasser&amp;volume=9&amp;issue=1&amp;publication_year=2014&amp;pages=171-181&amp;pmid=24385147&amp;doi=10.1038/nprot.2014.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<cite>
Satija R., Farrell J. A., Gennert D., Schier A. F., Regev A. (2015). Spatial Reconstruction of Single-Cell Gene Expression Data. Nat. Biotechnol.
33 (5), 495–502. 10.1038/nbt.3192
</cite> [<a href="https://doi.org/10.1038/nbt.3192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4430369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25867923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Spatial%20Reconstruction%20of%20Single-Cell%20Gene%20Expression%20Data&amp;author=R.%20Satija&amp;author=J.%20A.%20Farrell&amp;author=D.%20Gennert&amp;author=A.%20F.%20Schier&amp;author=A.%20Regev&amp;volume=33&amp;issue=5&amp;publication_year=2015&amp;pages=495-502&amp;pmid=25867923&amp;doi=10.1038/nbt.3192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<cite>
Saviano A., Henderson N. C., Baumert T. F. (2020). Single-cell Genomics and Spatial Transcriptomics: Discovery of Novel Cell States and Cellular Interactions in Liver Physiology and Disease Biology. J. Hepatology
73 (5), 1219–1230. 10.1016/j.jhep.2020.06.004
</cite> [<a href="https://doi.org/10.1016/j.jhep.2020.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7116221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32534107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Hepatology&amp;title=Single-cell%20Genomics%20and%20Spatial%20Transcriptomics:%20Discovery%20of%20Novel%20Cell%20States%20and%20Cellular%20Interactions%20in%20Liver%20Physiology%20and%20Disease%20Biology&amp;author=A.%20Saviano&amp;author=N.%20C.%20Henderson&amp;author=T.%20F.%20Baumert&amp;volume=73&amp;issue=5&amp;publication_year=2020&amp;pages=1219-1230&amp;pmid=32534107&amp;doi=10.1016/j.jhep.2020.06.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<cite>
Shah S., Takei Y., Wen Z., Lubeck E., Yun J., Linus Eng C.-H., et al.  (2018). Christoph Karp, Eric J. Liaw, Mina Amin, and Long Cai. Dynamics and Spatial Genomics of the Nascent Transcriptome by Intron Seqfish. Cell.
174 (2), 363–376. e16. 10.1016/j.cell.2018.05.035
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.05.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6046268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29887381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.&amp;title=Christoph%20Karp,%20Eric%20J.%20Liaw,%20Mina%20Amin,%20and%20Long%20Cai.%20Dynamics%20and%20Spatial%20Genomics%20of%20the%20Nascent%20Transcriptome%20by%20Intron%20Seqfish&amp;author=S.%20Shah&amp;author=Y.%20Takei&amp;author=Z.%20Wen&amp;author=E.%20Lubeck&amp;author=J.%20Yun&amp;volume=174&amp;issue=2&amp;publication_year=2018&amp;pages=363-376&amp;pmid=29887381&amp;doi=10.1016/j.cell.2018.05.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<cite>
Ståhl Patrik L., Salmén F., Vickovic S., Lundmark A., José F. N., Magnusson J., et al.  (2016). Visualization and Analysis of Gene Expression in Tissue Sections by Spatial Transcriptomics. Science
353 (6294), 78–82. 10.1126/science.aaf2403
</cite> [<a href="https://doi.org/10.1126/science.aaf2403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27365449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Visualization%20and%20Analysis%20of%20Gene%20Expression%20in%20Tissue%20Sections%20by%20Spatial%20Transcriptomics&amp;author=L.%20St%C3%A5hl%20Patrik&amp;author=F.%20Salm%C3%A9n&amp;author=S.%20Vickovic&amp;author=A.%20Lundmark&amp;author=F.%20N.%20Jos%C3%A9&amp;volume=353&amp;issue=6294&amp;publication_year=2016&amp;pages=78-82&amp;pmid=27365449&amp;doi=10.1126/science.aaf2403&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<cite>
Strell C., Hilscher M. M., Laxman N., Svedlund J., Wu C., Yokota C., et al.  (2019). Placing Rna in Context and Space - Methods for Spatially Resolved Transcriptomics. FEBS J.
286 (8), 1468–1481. 10.1111/febs.14435
</cite> [<a href="https://doi.org/10.1111/febs.14435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29542254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&amp;title=Placing%20Rna%20in%20Context%20and%20Space%20-%20Methods%20for%20Spatially%20Resolved%20Transcriptomics&amp;author=C.%20Strell&amp;author=M.%20M.%20Hilscher&amp;author=N.%20Laxman&amp;author=J.%20Svedlund&amp;author=C.%20Wu&amp;volume=286&amp;issue=8&amp;publication_year=2019&amp;pages=1468-1481&amp;pmid=29542254&amp;doi=10.1111/febs.14435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<cite>
Sun S., Hood M., Scott L., Peng Q., Mukherjee S., Tung J., et al.  (2017). Differential Expression Analysis for Rnaseq Using Poisson Mixed Models. Nucleic Acids Res.
45 (11), e106. 10.1093/nar/gkx204
</cite> [<a href="https://doi.org/10.1093/nar/gkx204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5499851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28369632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Differential%20Expression%20Analysis%20for%20Rnaseq%20Using%20Poisson%20Mixed%20Models&amp;author=S.%20Sun&amp;author=M.%20Hood&amp;author=L.%20Scott&amp;author=Q.%20Peng&amp;author=S.%20Mukherjee&amp;volume=45&amp;issue=11&amp;publication_year=2017&amp;pages=e106&amp;pmid=28369632&amp;doi=10.1093/nar/gkx204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<cite>
Sun S., Zhu J., Mozaffari S., Ober C., Chen M., Zhou X. (2019). Heritability Estimation and Differential Analysis of Count Data with Generalized Linear Mixed Models in Genomic Sequencing Studies. Bioinformatics
35 (3), 487–496. 10.1093/bioinformatics/bty644
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6361238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30020412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Heritability%20Estimation%20and%20Differential%20Analysis%20of%20Count%20Data%20with%20Generalized%20Linear%20Mixed%20Models%20in%20Genomic%20Sequencing%20Studies&amp;author=S.%20Sun&amp;author=J.%20Zhu&amp;author=S.%20Mozaffari&amp;author=C.%20Ober&amp;author=M.%20Chen&amp;volume=35&amp;issue=3&amp;publication_year=2019&amp;pages=487-496&amp;pmid=30020412&amp;doi=10.1093/bioinformatics/bty644&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<cite>
Sun S., Zhu J., Zhou X. (2020). Statistical Analysis of Spatial Expression Patterns for Spatially Resolved Transcriptomic Studies. Nat. Methods
17 (2), 193–200. 10.1038/s41592-019-0701-7
</cite> [<a href="https://doi.org/10.1038/s41592-019-0701-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7233129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31988518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Statistical%20Analysis%20of%20Spatial%20Expression%20Patterns%20for%20Spatially%20Resolved%20Transcriptomic%20Studies&amp;author=S.%20Sun&amp;author=J.%20Zhu&amp;author=X.%20Zhou&amp;volume=17&amp;issue=2&amp;publication_year=2020&amp;pages=193-200&amp;pmid=31988518&amp;doi=10.1038/s41592-019-0701-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<cite>
Tasic B., Yao Z., Graybuck L. T., Smith K. A., Nguyen T. N., Bertagnolli D., et al.  (2018). Shared and Distinct Transcriptomic Cell Types across Neocortical Areas. Nature
563 (7729), 72–78. 10.1038/s41586-018-0654-5
</cite> [<a href="https://doi.org/10.1038/s41586-018-0654-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6456269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30382198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Shared%20and%20Distinct%20Transcriptomic%20Cell%20Types%20across%20Neocortical%20Areas&amp;author=B.%20Tasic&amp;author=Z.%20Yao&amp;author=L.%20T.%20Graybuck&amp;author=K.%20A.%20Smith&amp;author=T.%20N.%20Nguyen&amp;volume=563&amp;issue=7729&amp;publication_year=2018&amp;pages=72-78&amp;pmid=30382198&amp;doi=10.1038/s41586-018-0654-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<cite>
Tim S., Satija R. (2019). Integrative Single-Cell Analysis. Nat. Rev. Genet.
20 (5), 257–272. 10.1038/s41576-019-0093-7
</cite> [<a href="https://doi.org/10.1038/s41576-019-0093-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30696980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Integrative%20Single-Cell%20Analysis&amp;author=S.%20Tim&amp;author=R.%20Satija&amp;volume=20&amp;issue=5&amp;publication_year=2019&amp;pages=257-272&amp;pmid=30696980&amp;doi=10.1038/s41576-019-0093-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<cite>
Wang T., Liu Y., Yin Q., Geng J., Chen J., Yin X., et al.  (2022). Enhancing Discoveries of Molecular Qtl Studies with Small Sample Size Using Summary Statistic Imputation. Briefings Bioinforma.
23 (1), bbab370. 10.1093/bib/bbab370
</cite> [<a href="https://doi.org/10.1093/bib/bbab370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34545927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinforma.&amp;title=Enhancing%20Discoveries%20of%20Molecular%20Qtl%20Studies%20with%20Small%20Sample%20Size%20Using%20Summary%20Statistic%20Imputation&amp;author=T.%20Wang&amp;author=Y.%20Liu&amp;author=Q.%20Yin&amp;author=J.%20Geng&amp;author=J.%20Chen&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pages=bbab370&amp;pmid=34545927&amp;doi=10.1093/bib/bbab370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<cite>
Wang X., Allen W. E., Wright Matthew A., Sylwestrak Emily L., Nikolay S., Sam V., et al.  (2018). Bava Felice-Alessio, and Deisseroth Karl. Three-Dimensional Intact-Tissue Sequencing of Single-Cell Transcriptional States. Science
361 (6400), eaat5691. 10.1126/science.aat5691
</cite> [<a href="https://doi.org/10.1126/science.aat5691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6339868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29930089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Bava%20Felice-Alessio,%20and%20Deisseroth%20Karl.%20Three-Dimensional%20Intact-Tissue%20Sequencing%20of%20Single-Cell%20Transcriptional%20States&amp;author=X.%20Wang&amp;author=W.%20E.%20Allen&amp;author=A.%20Wright%20Matthew&amp;author=L.%20Sylwestrak%20Emily&amp;author=S.%20Nikolay&amp;volume=361&amp;issue=6400&amp;publication_year=2018&amp;pages=eaat5691&amp;pmid=29930089&amp;doi=10.1126/science.aat5691&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<cite>
Xia C., Fan J., George E., Junjie H., Zhuang X. (2019). Spatial Transcriptome Profiling by Merfish Reveals Subcellular Rna Compartmentalization and Cell Cycle-dependent Gene Expression. Proc. Natl. Acad. Sci. U. S. A.
116 (39), 19490–19499. 10.1073/pnas.1912459116
</cite> [<a href="https://doi.org/10.1073/pnas.1912459116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6765259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31501331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Spatial%20Transcriptome%20Profiling%20by%20Merfish%20Reveals%20Subcellular%20Rna%20Compartmentalization%20and%20Cell%20Cycle-dependent%20Gene%20Expression&amp;author=C.%20Xia&amp;author=J.%20Fan&amp;author=E.%20George&amp;author=H.%20Junjie&amp;author=X.%20Zhuang&amp;volume=116&amp;issue=39&amp;publication_year=2019&amp;pages=19490-19499&amp;pmid=31501331&amp;doi=10.1073/pnas.1912459116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<cite>
Yang J., Lee S. H., Goddard M. E., Visscher P. M. (2011). Gcta: a Tool for Genome-wide Complex Trait Analysis. Am. J. Hum. Genet.
88 (1), 76–82. 10.1016/j.ajhg.2010.11.011
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2010.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3014363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&amp;title=Gcta:%20a%20Tool%20for%20Genome-wide%20Complex%20Trait%20Analysis&amp;author=J.%20Yang&amp;author=S.%20H.%20Lee&amp;author=M.%20E.%20Goddard&amp;author=P.%20M.%20Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;doi=10.1016/j.ajhg.2010.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<cite>
Zheng G. X. Y., Terry J. M., Belgrader P., Paul R., Zachary W. B., Ryan W., et al.  (2017). Massively Parallel Digital Transcriptional Profiling of Single Cells. Nat. Commun.
8, 14049. 10.1038/ncomms14049
</cite> [<a href="https://doi.org/10.1038/ncomms14049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5241818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Massively%20Parallel%20Digital%20Transcriptional%20Profiling%20of%20Single%20Cells&amp;author=G.%20X.%20Y.%20Zheng&amp;author=J.%20M.%20Terry&amp;author=P.%20Belgrader&amp;author=R.%20Paul&amp;author=W.%20B.%20Zachary&amp;volume=8&amp;publication_year=2017&amp;pages=14049&amp;pmid=28091601&amp;doi=10.1038/ncomms14049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<cite>
Zhu J., Sun S., Zhou X. (2021). Spark-x: Non-parametric Modeling Enables Scalable and Robust Detection of Spatial Expression Patterns for Large Spatial Transcriptomic Studies. Genome Biol.
22 (1), 184. 10.1186/s13059-021-02404-0
</cite> [<a href="https://doi.org/10.1186/s13059-021-02404-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8218388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34154649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Spark-x:%20Non-parametric%20Modeling%20Enables%20Scalable%20and%20Robust%20Detection%20of%20Spatial%20Expression%20Patterns%20for%20Large%20Spatial%20Transcriptomic%20Studies&amp;author=J.%20Zhu&amp;author=S.%20Sun&amp;author=X.%20Zhou&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=184&amp;pmid=34154649&amp;doi=10.1186/s13059-021-02404-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All source code used in our experiments and the processed data has been deposited at <a href="https://sqsun.github.io/software.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sqsun.github.io/software.html</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Frontiers in Genetics are provided here courtesy of <strong>Frontiers Media SA</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.3389/fgene.2022.893522" class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/fgene-13-893522.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.8&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/9181802/" data-citation-style="nlm" data-download-format-link="/resources/citations/9181802/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC9181802%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC9181802/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC9181802/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC9181802/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/35692845/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC9181802/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/35692845/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC9181802/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/9181802/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="abstract1" href="#abstract1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s1" href="#s1" class="usa-in-page-nav__link usa-current" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="1 Introduction">1 Introduction</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s2" href="#s2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="2 Results">2 Results</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s3" href="#s3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="3 Methods and Materials">3 Methods and Materials</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s4" href="#s4" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="4 Discussion">4 Discussion</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s5" href="#s5" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Data Availability Statement">Data Availability Statement</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s6" href="#s6" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Author Contributions">Author Contributions</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s7" href="#s7" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Funding">Funding</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s8" href="#s8" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Conflict of Interest">Conflict of Interest</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="s9" href="#s9" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Publisher’s Note">Publisher’s Note</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list1" href="#ref-list1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="0KXlMB2Ia6DJStkiFYTcu6xuyLN5zIFUImguKFisTWIxpNI48XIkMAZG2435PDt3">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    

<script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.29.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.29.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn" style="position: fixed; visibility: visible; cursor: pointer; border: none; background-color: transparent; padding: 0px; margin: 0px; bottom: 126px; right: -1px; width: 52px;"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div><img alt="" src="https://pmc.ncbi.nlm.nih.gov/stat?browserheight=850&amp;browserwidth=865&amp;canscroll_x=false&amp;canscroll_y=true&amp;colorDepth=24&amp;connection_effectivetype=4g&amp;connection_type=&amp;cookieSize=1024&amp;cookieenabled=true&amp;duration._ad93_.scrollInfo=0&amp;duration._adda93_.scrollInfo=0&amp;duration.abstract1.scrollInfo=111&amp;duration.historyarticle-meta1.scrollInfo=111&amp;duration.kwd-group1.scrollInfo=16&amp;duration.ref-list1.scrollInfo=0&amp;duration.ref-list1_sec2.scrollInfo=0&amp;duration.s1.scrollInfo=109&amp;duration.s2-1.scrollInfo=26&amp;duration.s2-2.scrollInfo=26&amp;duration.s2-3.scrollInfo=0&amp;duration.s2.scrollInfo=108&amp;duration.s3-1.scrollInfo=0&amp;duration.s3-2.scrollInfo=0&amp;duration.s3-3.scrollInfo=0&amp;duration.s3.scrollInfo=0&amp;duration.s4.scrollInfo=0&amp;duration.s5.scrollInfo=0&amp;duration.s6.scrollInfo=0&amp;duration.s7.scrollInfo=0&amp;duration.s8.scrollInfo=0&amp;duration.s9.scrollInfo=0&amp;eventid=2&amp;is_browser_supported=true&amp;jsevent=unload&amp;jsloadtime=&amp;jsperf_basePage=71&amp;jsperf_connect=110&amp;jsperf_dns=5&amp;jsperf_navTimingNetworkMS=932&amp;jsperf_navTimingPageMS=1246&amp;jsperf_navTimingRedirectCount=0&amp;jsperf_navTimingType=navigate&amp;jsperf_navType=0&amp;jsperf_redirectCount=0&amp;jsperf_ttfb=245&amp;language_code=en-US&amp;myncbi_signed_in=false&amp;ncbi_algorithm=&amp;ncbi_app=cloudpmc-viewer&amp;ncbi_db=pmc&amp;ncbi_domain=frontgene&amp;ncbi_feature=associated_data&amp;ncbi_featured_srcdb=&amp;ncbi_nwds=&amp;ncbi_pcid=journal&amp;ncbi_pdid=article&amp;ncbi_phid=118E307683B96893023076007A764FF0.m_1&amp;ncbi_program=&amp;ncbi_timeonpage=&amp;ncbi_timesinceload=118695&amp;ncbi_timesincenavstart=120083&amp;ncbi_type=fulltext&amp;pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal&amp;pinger_in_iframe=false&amp;raw_ga=false&amp;screenavailheight=1053&amp;screenavailwidth=1848&amp;screenheight=1080&amp;screenwidth=1920&amp;server=pmc.ncbi.nlm.nih.gov&amp;sgSource=native&amp;sgversion=0.42.0&amp;sgversion_hotfix=0&amp;sgversion_major=0&amp;sgversion_minor=42&amp;spa_index=0&amp;unload.mutator_count=1&amp;x.maxscroll=0&amp;x.pagesize=10&amp;y.maxscroll=27&amp;y.pagesize=269" style="display: none;"></body></html>